// blockly/msg/be.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Дадаць каментарый";
Blockly.Msg["ALT_KEY"] = "Alt";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Немагчыма выдаліць пераменную '%1', паколькі яна з’яўляецца часткай вызначэння функцыі '%2'";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Змяніць значэння:";
Blockly.Msg["CHROME_OS"] = "ChromeOS";  // untranslated
Blockly.Msg["CLEAN_UP"] = "Выраўнаваць блокі";
Blockly.Msg["CLOSE"] = "Close";  // untranslated
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Collapsed blocks contain warnings.";  // untranslated
Blockly.Msg["COLLAPSE_ALL"] = "Згарнуць блокі";
Blockly.Msg["COLLAPSE_BLOCK"] = "Згарнуць блок";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "колер 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "колер 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "суадносіны";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "змяшаць";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Змешвае два колеры разам у паказаных суадносінах (0.0 - 1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://be.wikipedia.org/wiki/Колер";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Выбраць колер з палітры.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "выпадковы колер";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Выбраць колер наўздагад.";
Blockly.Msg["COLOUR_RGB_BLUE"] = "сіні";
Blockly.Msg["COLOUR_RGB_GREEN"] = "зялёны";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "чырвоны";
Blockly.Msg["COLOUR_RGB_TITLE"] = "колер з";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Стварыць колер з названымі ўзроўнямі чырвонага, зялёнага і сіняга. Усе значэння павінны быць ад 0 да 100.";
Blockly.Msg["COMMAND_KEY"] = "Command";  // untranslated
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "перарваць цыкл";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "прадоўжыць з наступнай ітэрацыі цыклу";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Перарваць выкананне цыкла.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Прапусціць рэшту гэтага цыкла і перайсці да выканання наступнай ітэрацыі.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Папярэджанне: гэты блок можа быць выкарыстаны толькі ў межах цыклу.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "для кожнага элемента %1 у спісе %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Для кожнага элемента ў спісе пераменная '%1' атрымлівае значэнне элемента, а затым выконваюцца пэўныя дзеянні.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "лічыць з %1 ад %2 да %3 праз %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Наяўная пераменная '%1' прымае значэнні ад пачатковага да канчатковага, улічваючы зададзены інтэрвал, і выконваюцца названыя блокі.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Дадайце ўмову да блоку 'калі'";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Дадаць канчатковую, ўсёабдымную ўмову ў блок 'калі'.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Дадайце, выдаліце або змяніце парадак секцыяў, каб пераналадзіць гэты блок 'калі'.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "інакш";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "інакш калі";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "калі";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Калі значэнне ісцінае, то выканаць пэўныя дзеянні.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Калі значэнне ісцінае, то выконваецца першы блок аператараў. У адваротным выпадку выконваецца другі блок аператараў.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Калі першае значэнне ісцінае, то выконваецца першы блок аператараў. У адваротным выпадку, калі другое значэнне ісцінае, то выконваецца другі блок аператараў.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Калі першае значэнне ісцінае, то выконваецца першы блок аператараў. У адваротным выпадку, калі другое значэнне ісцінае, то выконваецца другі блок аператараў. Калі ні адно з значэнняў не з’яўляецца ісціным, то выконваецца апошні блок аператараў.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://be.wikipedia.org/wiki/Цыкл_(праграмаванне)";
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "выканаць";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "паўтарыць %1 разоў";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Выканаць пэўныя дзеянні некалькі разоў.";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "паўтараць, пакуль не";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "паўтараць, пакуль";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Пакуль значэнне ілжывае, выконваць пэўныя дзеянні.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Пакуль значэнне ісцінае, выконваць пэўныя дзеянні.";
Blockly.Msg["CONTROL_KEY"] = "Control";  // untranslated
Blockly.Msg["COPY_SHORTCUT"] = "Copy";  // untranslated
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Cut";  // untranslated
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Выдаліць усе блокі %1?";
Blockly.Msg["DELETE_BLOCK"] = "Выдаліць блок";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Выдаліць пераменную '%1'";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Выдаліць %1 выкарыстання пераменнай '%2'?";
Blockly.Msg["DELETE_X_BLOCKS"] = "Выдаліць %1 блокаў";
Blockly.Msg["DIALOG_CANCEL"] = "Скасаваць";
Blockly.Msg["DIALOG_OK"] = "Добра";
Blockly.Msg["DISABLE_BLOCK"] = "Адключыць блок";
Blockly.Msg["DUPLICATE_BLOCK"] = "Дубляваць";
Blockly.Msg["DUPLICATE_COMMENT"] = "Duplicate Comment";  // untranslated
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Edit Block contents";  // untranslated
Blockly.Msg["ENABLE_BLOCK"] = "Уключыць блок";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Разгарнуць блокі";
Blockly.Msg["EXPAND_BLOCK"] = "Разгарнуць блок";
Blockly.Msg["EXTERNAL_INPUTS"] = "Знешнія ўваходы";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Даведка";
Blockly.Msg["HELP_PROMPT"] = "Press %1 for help on keyboard controls.";  // untranslated
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Убудаваныя ўваходы";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Use the arrow keys to move, then %1 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Copied. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Cut. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Hold %1 and use arrow keys to move freely, then %2 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "стварыць пусты спіс";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Вяртае спіс даўжыні 0, які не ўтрымлівае дадзеных";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "спіс";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Дадайце, выдаліце, перастаўце элементы для пераробкі блока спісу.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "стварыць спіс з";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Дадае элемент да спісу.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Стварае спіс з любым лікам элементаў.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "першы";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# з канца";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_GET"] = "узяць";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "узяць і выдаліць";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "апошні";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "адвольны";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "выдаліць";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Вяртае першы элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Вяртае элемент ва ўказанай пазіцыі спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Вяртае апошні элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Вяртае выпадковы элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Видаляє і повертає перший елемент списку.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Выдаляе і вяртае элемент ва ўказанай пазіцыі спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Видаляє і повертає апошні элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Видаляє і повертає выпадковы элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Видаляє першы элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Видаляє элемент ва ўказанай пазіцыі спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Видаляє апошні элемент спіса.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Видаляє выпадковы элемент спіса.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "па # з канца";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "па #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "па апошні";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "узяць падспіс з першага";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "узяць падспіс з # з канца";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "узяць падспіс з #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Стварае копію ўказанай частцы спісу.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 - апошні элемент.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 - першы элемент.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "знайсці першае ўваходжанне элемента";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "знайсці апошняе ўваходжанне элемента";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Вяртае нумар пазіцыі першага/апошняга ўваходжання элемента ў спісе. Вяртае %1, калі элемент не знойдзены.";
Blockly.Msg["LISTS_INLIST"] = "у спісе";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 пусты";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Вяртае значэнне ісціна, калі спіс пусты.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "даўжыня %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Вяртае даўжыню спісу.";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "стварыць спіс з элемента %1, паўтаральнага %2 разоў";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Стварае спіс, які складаецца з зададзенага ліку копій элемента.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "змяніць парадак на зваротны %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Змяніць парадак спісу на адваротны.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "=";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "уставіць у";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "прысабечыць";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Устаўляе элемент у пачатак спіса.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Устаўляе элемент ва ўказанай пазіцыі спіса.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Дадае элемент у канец спісу.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Устаўляе элемент у выпадковае месца ў спісе.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Прысабечвае значэнне элемента спісу.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Прысабечвае значэнне элемента ва ўказанай пазіцыі спіса.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Прысабечвае значэнне элемента спісу.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Прысабечвае значэнне элемента спісу.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "па ўзрастанні";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "па спаданню";
Blockly.Msg["LISTS_SORT_TITLE"] = "сартаваць %1 %2 %3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Сартаваць копію спісу.";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "па алфавіце, без уліку рэгістра";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "лікавая";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "па алфавіце";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "зрабіць спіс з тэксту";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "сабраць тэкст са спісу";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Злучае спіс тэкстаў у адзін тэкст з падзельнікамі.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Разбівае тэкст у спіс тэкстаў па падзельнікам.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "з падзельнікам";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "хлусня";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Вяртае значэнне ісціна або хлусня.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "ісціна";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://be.wikipedia.org/wiki/Няроўнасць";
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Вяртае станоўчае значэнне, калі абодва ўводы роўныя адзін аднаму.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Вяртае значэнне ісціна, калі першая ўстаўка больш другой.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Вяртае значэнне ісціна, калі першая ўстаўка больш або роўная другой.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Вяртае станоўчае значэнне, калі першы ўвод менш другога.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Вяртае значэнне ісціна, калі першая ўстаўка менш або роўная другі.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Вяртае станоўчае значэнне, калі ўводы не роўныя.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "не %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Вяртае значэнне ісціна, калі устаўка ілжывая. Вяртае значэнне хлусня, калі устаўка ісцінная.";
Blockly.Msg["LOGIC_NULL"] = "нішто";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Вяртае нішто.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "і";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "ці";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Вяртае значэнне ісціна, калі абедзве ўстаўкі ісцінныя.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Вяртае значэнне ісціна, калі хаця б адна з уставак ісцінная.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "абраць па";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "калі хлусня";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "калі ісціна";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Правярае ўмову выбару. Калі ўмова ісцінная, вяртае першае значэнне, у адваротным выпадку вяртае другое значэнне.";
Blockly.Msg["MAC_OS"] = "macOS";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://be.wikipedia.org/wiki/Арыфметыка";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Вяртае суму двух лікаў.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Вяртае дзель ад дзялення першага ліку на другое.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Вяртае рознасць двух лікаў.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Вяртае твор двух лікаў.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Вяртае першы лік, узведзены ў ступень другога ліку.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 of X:%1 Y:%2";  // untranslated
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Return the arctangent of point (X, Y) in degrees from -180 to 180.";  // untranslated
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://be.wikipedia.org/wiki/Ідыёма (праграмаванне)";
Blockly.Msg["MATH_CHANGE_TITLE"] = "павялічыць %1 на %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Дадае лік да пераменнай '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://be.wikipedia.org/wiki/Матэматычная_канстанта";
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Вяртае адну з распаўсюджаных канстант: π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), або ∞ (бясконцасць).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "абмежаваць %1 знізу %2 зверху %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Абмяжоўвае лік ніжняй і верхняй межамі (уключна).";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "дзеліцца на";
Blockly.Msg["MATH_IS_EVEN"] = "цотнае";
Blockly.Msg["MATH_IS_NEGATIVE"] = "адмоўнае";
Blockly.Msg["MATH_IS_ODD"] = "няцотнае";
Blockly.Msg["MATH_IS_POSITIVE"] = "станоўчае";
Blockly.Msg["MATH_IS_PRIME"] = "простае";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Правярае, ці з’яўляецца лік цотным, няцотным, простым, цэлым, станоўчым, адмоўным ці яно кратнае пэўнаму ліку. Вяртае значэнне ісціна або хлусня.";
Blockly.Msg["MATH_IS_WHOLE"] = "цэлае";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://be.wikipedia.org/wiki/Дзяленне_з_астаткам";
Blockly.Msg["MATH_MODULO_TITLE"] = "астатак ад %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Вяртае астатак ад дзялення двух лікаў.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://be.wikipedia.org/wiki/Лік";
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Лік.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "сярэдняе арыфметычнае спісу";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "найбольшы ў спісе";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "медыяна спіса";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "найменшы ў спісе";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "моды спіса";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "выпадковы элемент спіса";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "стандартнае адхіленне спіса";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "сума спіса";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Вяртае сярэдняе арыфметычнае спісу.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Вяртае найбольшы лік спісу.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Вяртае медыяну спісуа.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Вяртае найменшы лік спісу.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Вяртае спіс найбольш часта сустракаемых элементаў спісу.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Вяртае выпадковы элемент спіса.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Вяртае стандартнае адхіленне спісу.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Вяртае суму ўсіх лікаў у спісе.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://be.wikipedia.org/wiki/Генератар_псевдослучайных_лікаў";
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "выпадковы лік ад 0 (уключна) да 1";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Вяртае выпадковы лік ад 0.0 (уключна) да 1.0.";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://be.wikipedia.org/wiki/Генератар_псевдослучайных_лікаў";
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "выпадковы цэлы лік ад %1 для %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Вяртае выпадковы лік паміж двума зададзенымі межамі (уключаючы і іх).";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://be.wikipedia.org/wiki/Акругленне";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "акругліць";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "акругліць да меншага";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "акругліць да большага";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Акругляе лік да большага ці меншага.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://be.wikipedia.org/wiki/Квадратны_корань";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "модуль";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "квадратны корань";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Вяртае модуль ліку.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Вяртае е ва ўказанай ступені.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Вяртае натуральны лагарыфм ліку.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Вяртае дзесятковы лагарыфм ліку.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Вяртае супрацьлеглы лік.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Вяртае 10 ва ўказанай ступені.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Вяртае квадратны корань ліку.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "cos";  // untranslated
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://be.wikipedia.org/wiki/Трыганаметрычныя_функцыі";
Blockly.Msg["MATH_TRIG_SIN"] = "sin";  // untranslated
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";  // untranslated
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Вяртае арккосінус у градусах.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Вяртае арксінус у градусах.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Вяртае арктангенс у градусах.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Вяртае косінус рага ў градусах.";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Вяртае сінус рага ў градусах.";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Вяртае тангенс рага ў градусах.";
Blockly.Msg["MOVE_BLOCK"] = "Move Block";  // untranslated
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Create colour variable...";  // untranslated
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Create number variable...";  // untranslated
Blockly.Msg["NEW_STRING_VARIABLE"] = "Create string variable...";  // untranslated
Blockly.Msg["NEW_VARIABLE"] = "Стварыць пераменную...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Новая назва пераменнай:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "New variable type:";  // untranslated
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "Option";  // untranslated
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Paste";  // untranslated
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "дазволіць аператары";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "з:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://be.wikipedia.org/wiki/Падпраграма";
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Выконвае вызначаную карыстальнікам функцыю '%1'.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://be.wikipedia.org/wiki/Падпраграма";
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Выконвае вызначаную карыстальнікам функцыю '%1' і ўжыць яе значэнне.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "з:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Can't run the user-defined function '%1' because the definition block is disabled.";  // untranslated
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Стварыць выклік '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Апішыце гэтую функцыю...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "выканаць нешта";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "каб";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Стварыць функцыю, якая не вяртае значэнне.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "вярнуць";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Стварыць функцыю, якая вяртае значэнне.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Папярэджанне: гэтая функцыя мае паўтаральныя параметры.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Вылучыць вызначэнне функцыі";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Калі першае значэнне ісцінае, вяртае другое значэнне.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Папярэджанне: гэты блок можа выкарыстоўвацца толькі ўнутры вызначэння функцыі.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "імя параметра:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Дадаць уваходны параметр ў функцыю.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "параметры";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Дадаць, выдаліць або змяніць парадак уваходных параметраў для гэтай функцыі.";
Blockly.Msg["REDO"] = "Паўтарыць";
Blockly.Msg["REMOVE_COMMENT"] = "Выдаліць каментар";
Blockly.Msg["RENAME_VARIABLE"] = "Перайменаваць пераменную...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Перайменаваць усе пераменныя '%1' да:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Code navigation";  // untranslated
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Editing";  // untranslated
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "General";  // untranslated
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "да %1 дадаць тэкст %2";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Дадаць тэкст да пераменнай '%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "у малыя літары";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "у Загалоўныя Пачатковыя Літары";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "у ВЯЛІКІХ ЛІТАРАЎ";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Вяртае копію тэксту з ВЯЛІКІМІ або малымі літарамі.";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "узяць першую літару";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "узяць літару # з канца";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "узяць літару #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "узяць апошнюю літару";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "узяць выпадковую літару";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "у тэксце %1 %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Вяртае літару ў названай пазіцыі.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "падлічыць колькасць %1 у %2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Падлічыць, колькі разоў урывак тэксту з’яўляецца ў іншым тэксце.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Дадаць элемент да тэксту.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "спалучыць";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Дадайце, выдаліце, перастаўце фрагменты для пераробкі тэкставага блока.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "па літару # з канца";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "па літару #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "па апошнюю літару";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "у тэксце";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "узяць падрадок з першай літары";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "узяць падрадок з літары # з канца";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "узяць падрадок з літары #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Вяртае указаную частку тэксту.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "знайсці першае ўваходжанне тэксту";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "знайсці апошняе ўваходжанне тэксту";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "у тэксце %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Вяртае нумар пазіцыі першага/апошняга ўваходжання першага тэксту ў другім. Вяртае %1, калі тэкст не знойдзены.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 пусты";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Вяртае значэнне ісціна, калі пададзены тэкст пусты.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "стварыць тэкст з";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Стварае фрагмент тэксту, аб'ядноўваючы любы лік элементаў.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "даўжыня %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Вяртае колькасць сімвалаў (уключаючы прабелы) у зададзеным тэксце.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "надрукаваць %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Друкуе тэкст, лік або іншы аб'ект.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Запытаць у карыстальніка лік.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Запытаць у карыстальніка тэкст.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "запытаць лік з падказкай";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "запытаць тэкст з падказкай";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "замяніць %1 на %2 у %3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Замяніць усе ўваходжанні некаторага тэксту іншым тэкстам.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "змяніць парадак на зваротны %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Мяняе парадак знакаў у тэксце на адваротны.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://be.wikipedia.org/wiki/Радковы_тып";
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Літара, слова або радок тэксту.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "абрэзаць прагалы з двух бакоў";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "абрэзаць прагалы злева";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "абрэзаць прагалы справа";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Вяртае копію тэксту з прагаламі, выдаленымі з аднаго або абодвух канцоў.";
Blockly.Msg["TODAY"] = "Сёння";
Blockly.Msg["UNDO"] = "Адкаціць";
Blockly.Msg["UNKNOWN"] = "Unknown";  // untranslated
Blockly.Msg["UNNAMED_KEY"] = "неназваны";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "элемент";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Стварыць блок \"прысабечыць\" для %1";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Вяртае значэнне гэтай пераменнай.";
Blockly.Msg["VARIABLES_SET"] = "прысабечыць %1 = %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Стварыць устаўку %1";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Прысабечвае пераменнай значэнне ўстаўкі.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Пераменная з назвай '%1' ужо існуе.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Назва пераменнай '%1' ужо выкарыстоўваецца іншай пераменнай тыпу '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "A variable named '%1' already exists as a parameter in the procedure '%2'.";  // untranslated
Blockly.Msg["WINDOWS"] = "Windows";  // untranslated
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Say something...";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
