// blockly/msg/br.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Ouzhpennañ un evezhiadenn";
Blockly.Msg["ALT_KEY"] = "Alt";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Ne ch'aller ket dilemel an argemmenn \"%1\" dre m'eo lod eus dielfennadur an arc'hwel \"%2\"";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Kemmañ an dalvoudenn :";
Blockly.Msg["CHROME_OS"] = "ChromeOS";  // untranslated
Blockly.Msg["CLEAN_UP"] = "Naetaat ar bloc'hoù";
Blockly.Msg["CLOSE"] = "Close";  // untranslated
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Kemennoù-diwall zo er bloc'hadoù pleget.";
Blockly.Msg["COLLAPSE_ALL"] = "Bihanaat ar bloc'hoù";
Blockly.Msg["COLLAPSE_BLOCK"] = "Bihanaat ar bloc'h";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "liv 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "liv 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "feur";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "meskañ";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "a gemmesk daou liv gant ur feur roet(0.0 - 1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://br.wikipedia.org/wiki/Liv";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Dibab ul liv diwar al livaoueg.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "liv dargouezhek";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Tennañ ul liv d'ar sord";
Blockly.Msg["COLOUR_RGB_BLUE"] = "glas";
Blockly.Msg["COLOUR_RGB_GREEN"] = "gwer";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "ruz";
Blockly.Msg["COLOUR_RGB_TITLE"] = "liv gant";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Krouiñ ul liv gant ar c'hementad spisaet a ruz, a wer hag a c'hlas. Etre 0 ha 100 e tle bezañ an holl dalvoudoù.";
Blockly.Msg["COMMAND_KEY"] = "Command";  // untranslated
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "Mont e-maez an adlañsañ";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "Kenderc'hel gant iteradur nevez ar rodell";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Mont e-maez ar boukl engronnus.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Dilezel ar peurrest eus ar boukl-mañ, ha kenderc'hel gant an iteradur da-heul.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Diwallit : ne c'hall ar bloc'h-mañ bezañ implijet nemet e-barzh ur boukl.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "evit pep elfenn %1 er roll %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Evit pep elfenn en ur roll, reiñ talvoud an elfenn d'an argemmenn '%1', ha seveniñ urzhioù zo da c'houde.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "kontañ gant %1 eus %2 da %3 dre %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Ober e doare ma kemero an argemmenn \"%1\" an talvoudennoù adalek niverenn an deroù betek niverenn an dibenn, en ur inkremantiñ an esaouenn, ha seveniñ an urzhioù spisaet.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Ouzhpennañ un amplegad d'ar bloc'h ma.";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Ouzhpennañ un amplegad dibenn lak-pep-tra d'ar bloc'h ma.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Ouzhpennañ, lemel pe adurzhiañ ar rannoù evit kefluniañ ar bloc'h ma.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "a-hend-all";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "mod all ma";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "ma";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Ma vez gwir un dalvoudenn, seveniñ urzhioù zo neuze.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Ma vez gwir un dalvoudenn, seveniñ ar c'henañ bloc'had urzhioù neuze. Anez seveniñ an eil bloc'had urzhioù.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Ma vez gwir an dalvoudenn gentañ, seveniñ ar c'hentañ bloc'had urzhioù neuze. Anez ma vez gwir an eil talvoudenn, seveniñ an eil bloc'had urzhioù.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Ma vez gwir an dalvoudenn gentañ, seveniñ ar c'hentañ bloc'had. Anez, ma vez gwir an eil talvoudenn, seveniñ an eil bloc'had urzhioù. Ma ne vez gwir talvoudenn ebet, seveniñ ar bloc'had diwezhañ a urzhioù.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://en.wikipedia.org/wiki/For_loop";  // untranslated
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "ober";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "adober %1 gwech";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Seveniñ urzhioù zo meur a wech";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "adober betek";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "adober keit ha ma";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Keit ha ma vez faos un dalvoudenn, seveniñ urzhioù zo neuze.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Keit ha ma vez gwir un dalvoudenn, seveniñ urzhioù zo neuze.";
Blockly.Msg["CONTROL_KEY"] = "Control";  // untranslated
Blockly.Msg["COPY_SHORTCUT"] = "Copy";  // untranslated
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Cut";  // untranslated
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Diverkañ an holl vloc'hoù %1 ?";
Blockly.Msg["DELETE_BLOCK"] = "Dilemel ar bloc'h";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Lemel an argemm '%1'";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Lemel %1 implij eus an argemm '%2' ?";
Blockly.Msg["DELETE_X_BLOCKS"] = "Dilemel %1 bloc'h";
Blockly.Msg["DIALOG_CANCEL"] = "Nullañ";
Blockly.Msg["DIALOG_OK"] = "Mat eo";
Blockly.Msg["DISABLE_BLOCK"] = "Diweredekaat ar bloc'h";
Blockly.Msg["DUPLICATE_BLOCK"] = "Eiladuriñ";
Blockly.Msg["DUPLICATE_COMMENT"] = "Eiladiñ an eilenn";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Edit Block contents";  // untranslated
Blockly.Msg["ENABLE_BLOCK"] = "Gweredekaat ar bloc'h";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Astenn ar bloc'hoù";
Blockly.Msg["EXPAND_BLOCK"] = "Astenn ar bloc'h";
Blockly.Msg["EXTERNAL_INPUTS"] = "Monedoù diavaez";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Skoazell";
Blockly.Msg["HELP_PROMPT"] = "Press %1 for help on keyboard controls.";  // untranslated
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Monedoù enlinenn";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Use the arrow keys to move, then %1 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Copied. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Cut. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Hold %1 and use arrow keys to move freely, then %2 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "krouiñ ur roll goullo";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Distreiñ ul listenn, 0 a hirder, n'eus enrolladenn ebet enni";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "roll";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Ouzhpennañ, lemel pe adurzhiañ ar rannoù evit kefluniañ ar bloc'h listenn-mañ.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "krouiñ ur roll gant";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Ouzhpennañ un elfenn d'ar roll";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Krouiñ ur roll gant un niver bennak a elfennoù.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "kentañ";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# adalek ar fin";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "tapout";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "tapout ha lemel";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "diwezhañ";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "dre zegouezh";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "lemel";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Distreiñ an elfenn gentañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Distreiñ an elfenn el lec'h meneget en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Distreiñ un elfenn diwezhañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Distreiñ un elfenn dre zegouezh en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Lemel ha distreiñ a ra an elfenn gentañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Lemel ha distreiñ a ra an elfenn el lec'h meneget en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Lemel ha distreiñ a ra an elfenn diwezhañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Lemel ha distreiñ a ra an elfenn dre zegouezh en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Lemel a ra an elfenn gentañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Lemel a ra an elfenn el lec'h meneget en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Lemel a ra an elfenn diwezhañ en ul listenn.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Lemel a ra un elfenn dre zegouezh en ul listenn.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "betek # adalek an dibenn";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "da #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "betek ar fin";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "Kaout an islistenn adalek an deroù";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "Kaout an islistenn adalek # adalek an dibenn";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "Kaout an islistenn adalek #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Krouiñ un eilad eus lodenn spisaet ul listenn.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 eo an elfenn gentañ.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 eo an elfenn gentañ.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "kavout reveziadenn gentañ un elfenn";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "kavout reveziadenn diwezhañ un elfenn";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Distreiñ meneger ar c'hentañ/an eil reveziadenn eus an elfenn en ul listenn. Distreiñ %1 ma n'eo ket kavet an destenn.";
Blockly.Msg["LISTS_INLIST"] = "el listenn";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 zo goullo";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Distreiñ gwir m'eo goullo al listenn.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "hirder %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Distreiñ hirder ul listenn.";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "Krouiñ ul listenn gant an elfenn %1 arreet %2 div wech";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Krouiñ ul listenn a c'hoarvez eus an dalvoudenn roet arreet an niver a wech meneget";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "eilpennañ %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Eilpennañ un eilskrid eus ur roll.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "evel";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "ensoc'hañ evel";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "termenañ";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Ensoc'hañ a ra an elfenn e deroù ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Ensoc'hañ a ra an elfenn el lec'h meneget en ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Ouzhpennañ a ra an elfenn e fin al listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Ensoc'hañ a ra an elfenn dre zegouezh en ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Termenañ a ra an elfenn gentañ en ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Termenañ a ra an elfenn el lec'h meneget en ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Termenañ a ra an elfenn diwezhañ en ul listenn.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Termenañ a ra un elfenn dre zegouezh en ul listenn.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "war gresk";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "war zigresk";
Blockly.Msg["LISTS_SORT_TITLE"] = "Rummañ%1,%2,%3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Rummañ un eilenn eus ar roll";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "Dre urzh al lizherenneg, hep derc'hel kont eus an direnneg";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "niverel";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "Dre urzh al lizherenneg";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "Krouiñ ul listenn diwar an destenn";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "Krouiñ un destenn diwar al listenn";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Bodañ ul listennad testennoù en ul listenn hepken, o tispartiañ anezho gant un dispartier.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Troc'hañ un destenn en ul listennad testennoù, o troc'hañ e pep dispartier.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "gant an dispartier";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "gaou";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Distreiñ pe gwir pe faos";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "gwir";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://en.wikipedia.org/wiki/Inequality_(mathematics)";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Distreiñ gwir m'eo par an daou voned.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Distreiñ gwir m'eo brasoc'h ar moned kentañ eget an eil.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Distreiñ gwir m'eo brasoc'h ar moned kentañ eget an eil pe par dezhañ.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Distreiñ gwir m'eo bihanoc'h ar moned kentañ eget an eil.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Distreiñ gwir m'eo bihanoc'h ar moned kentañ eget an eil pe m'eo par dezhañ.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Distreiñ gwir ma n'eo ket par an daou voned.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "nann %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Distreiñ gwir m'eo faos ar moned. Distreiñ faos m'eo gwir ar moned.";
Blockly.Msg["LOGIC_NULL"] = "Null";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Distreiñ null.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "ha(g)";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "pe";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Distreiñ gwir m'eo gwir an da daou voned.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Distreiñ gwir m'eo gwir unan eus an daou voned da nebeutañ.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "amprouad";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "m'eo gaou";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "m'eo gwir";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Gwiriañ an amplegad e 'prouad'. M'eo gwir an amplegad, distreiñ an dalvoudenn 'm'eo gwir'; anez distreiñ ar moned 'm'eo faos'.";
Blockly.Msg["MAC_OS"] = "macOS";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://br.wikipedia.org/wiki/Aritmetik";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Distreiñ sammad daou niver.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Distreiñ rannad daou niver.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Distreiñ diforc'h daou niver";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Distreiñ liesad daou niver.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Distreiñ an niver kentañ lakaet dindan gallouter an eil niver.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://fr.wikipedia.org/wiki/Atan2";
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 eus X:%1 Y:%2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Adkas a ra ark tangent ar poent (X, Y) e derezioù etre -180 ha 180";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "kemmañ %1 gant %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Ouzhpennañ un niver d'an argemmenn '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://en.wikipedia.org/wiki/Mathematical_constant";  // untranslated
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Distreiñ unan eus digemmennoù red : π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), or ∞ (anvevenn).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "destrizhañ %1 etre %2 ha %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Destrizhañ un niver da vezañ etre ar bevennoù spisaet (enlakaet)";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "a zo rannadus dre";
Blockly.Msg["MATH_IS_EVEN"] = "zo par";
Blockly.Msg["MATH_IS_NEGATIVE"] = "a zo negativel";
Blockly.Msg["MATH_IS_ODD"] = "zo ampar";
Blockly.Msg["MATH_IS_POSITIVE"] = "a zo pozitivel";
Blockly.Msg["MATH_IS_PRIME"] = "zo kentañ";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Gwiriañ m'eo par, anpar, kentañ, muiel, leiel un niverenn pe ma c'haller rannañ anezhi dre un niver roet zo. Distreiñ gwir pe faos.";
Blockly.Msg["MATH_IS_WHOLE"] = "zo anterin";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://en.wikipedia.org/wiki/Modulo_operation";  // untranslated
Blockly.Msg["MATH_MODULO_TITLE"] = "rest eus %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Distreiñ dilerc'h rannadur an div niver";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://br.wikipedia.org/wiki/Niver";
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Un niver.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "Keitat al listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "Uc'hegenn al listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "Kreizad al listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "Izegenn al listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "modoù stankañ el listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "Elfennn eus al listenn tennet d'ar sord";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "forc'had standart eus al listenn";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "Sammad al listenn";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Distreiñ keitad (niveroniel) an talvoudennoù niverel el listenn.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Distreiñ an niver brasañ el listenn.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Distreiñ an niver kreiz el listenn";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Distreiñ an niver bihanañ el listenn";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Distreiñ ul listennad elfennoù stankoc'h el listenn.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Distreiñ un elfenn zargouezhek el listenn";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Distreiñ forc'had standart al listenn.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Distreiñ sammad an holl niveroù zo el listenn.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "Rann dargouezhek";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Distreiñ ur rann dargouezhek etre 0.0 (enkaelat) hag 1.0 (ezkaelat).";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "anterin dargouezhek etre %1 ha %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Distreiñ un anterin dargouezhek etre an div vevenn spisaet, endalc'het.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "Rontaat";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "Rontaat dindan";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "Rontaat a-us";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Rontaat un niver dindan pe a-us";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://br.wikipedia.org/wiki/Square_root";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "dizave";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "gwrizienn garrez";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Distreiñ talvoud dizave un niver.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Distreiñ galloudad un niver.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Distreiñ logaritm naturel un niver";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Distreiñ logaritm diazez 10 un niver";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Distreiñ enebad un niver";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Distreiñ 10 da c'halloudad un niver.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Distreiñ gwrizienn garrez un niver";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "cos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://br.wikipedia.org/wiki/Trigonometric_functions";
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Distreiñ ark kosinuz un niver";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Distreiñ ark sinuz un niver";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Distreiñ ark tangent un niver";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Distreiñ kosinuz ur c'horn (ket e radianoù)";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Distreiñ sinuz ur c'horn (ket e radianoù)";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Distreiñ tangent ur c'horn (ket e radianoù).";
Blockly.Msg["MOVE_BLOCK"] = "Move Block";  // untranslated
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Krouiñ un argemmenn liv...";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Krouiñ un argemm niver...";
Blockly.Msg["NEW_STRING_VARIABLE"] = "Krouiñ un argemmenn chadenn...";
Blockly.Msg["NEW_VARIABLE"] = "Krouiñ un argemmenn nevez...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Anv an argemmenn nevez :";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Doare nevez a argemm:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "Option";  // untranslated
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Paste";  // untranslated
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "aotren an disklêriadurioù";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "gant :";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Seveniñ an arc'hwel '%1' termenet gant an implijer.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Seveniñ an arc'hwel '%1' termenet gant an implijer hag implijout e zisoc'h.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "gant :";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Can't run the user-defined function '%1' because the definition block is disabled.";  // untranslated
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Krouiñ '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Deskrivañ an arc'hwel-mañ...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "ober un dra bennak";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "da";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Krouiñ un arc'hwel hep mont er-maez.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "distreiñ";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Kouiñ un arc'hwel gant ur mont er-maez";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Diwallit : an arc'hwel-mañ en deus arventennoù eiladet.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Dreislinennañ termenadur an arc'hwel";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Ma'z eo gwir un dalvoudenn, distreiñ un eil talvoudenn neuze.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Diwallit : Gallout a rafe ar bloc'h bezañ implijet e termenadur un arc'hwel hepken.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "Anv ar moned";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Ouzhpennañ ur moned d'an arc'hwel.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "Monedoù";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Ouzhpennañ, lemel, pe adkempenn monedoù an arc'hwel-mañ.";
Blockly.Msg["REDO"] = "Adober";
Blockly.Msg["REMOVE_COMMENT"] = "Lemel an evezhiadenn kuit";
Blockly.Msg["RENAME_VARIABLE"] = "Adenvel an argemmenn...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Adenvel an holl argemmennoù '%1' e :";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Code navigation";  // untranslated
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Editing";  // untranslated
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "General";  // untranslated
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "Ouzhpennañ an destenn %2 da %1";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Ouzhpennañ testenn d'an argemmenn'%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "e lizherennoù bihan";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "Gant Ur Bennlizherenn E Deroù Pep Ger";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "e PENNLIZHERENNOÙ";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Distreiñ un eilenn eus an eilenn en un direnneg all";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "tapout al lizherenn gentañ";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "Kaout al lizherenn # adalek an dibenn.";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "Kaout al lizherenn #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "tapout al lizherenn ziwezhañ";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "Kaout ul lizherenn dre zegouezh";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "en destenn %1 %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Distreiñ al lizherenn d'al lec'h spisaet.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "niver %1 war %2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Kontañ pet gwech e c'hoarvez un destenn bennak en un destenn bennak all.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Ouzhpennañ un elfenn d'an destenn.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "stagañ";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Ouzhpennañ, lemel pe adurzhiañ ar rannoù evit kefluniañ ar bloc'h testenn-mañ.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "Betek al lizherenn # adalek an dibenn.";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "betek al lizherenn #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "d'al lizherenn diwezhañ";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "en destenn";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "Kaout an ischadenn adalek al lizherenn gentañ";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "Kaout an ischadenn adalek al lizherenn # betek an dibenn";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "Kaout an ischadenn adalek al lizherenn #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Distreiñ un tamm spisaet eus an destenn.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "kavout reveziadenn gentañ an destenn";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "kavout reveziadenn diwezhañ an destenn";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "en destenn %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Distreiñ meneger ar c'hentañ/an eil reveziadenn eus ar chadenn gentañ en eil chadenn. Distreiñ %1 ma n'eo ket kavet ar chadenn.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 zo goullo";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Adkas gwir m'eo goullo an destenn roet.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "krouiñ un destenn gant";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Krouit un tamm testenn en ur gevelstrollañ un niver bennak a elfennoù";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "hirder %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Distreiñ an niver a lizherennoù(en ur gontañ an esaouennoù e-barzh) en destenn roet.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "moullañ %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Moullañ an destenn, an niverenn pe un dalvoudenn spisaet all";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Goulenn un niver gant an implijer.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Goulenn un destenn gant an implijer.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "pedadenn evit un niver gant ur c'hemennad";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "goulenn un destenn gant ur gemennadenn";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "erlec'hiañ %1 gant %2 e %3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Erlec'hiañ holl reveziadennoù un destenn bennak gant un destenn all.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "eilpennañ %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Eilpennañ urzh an arouezennoù en destenn.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://en.wikipedia.org/wiki/String_(computer_science)";  // untranslated
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Ul lizherenn, ur ger pe ul linennad testenn.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "Lemel an esaouennoù en daou du";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "Lemel an esaouennoù eus an tu kleiz";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "Lemel an esaouennoù eus an tu dehou";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Distreiñ un eilenn eus an destenn gant an esaouennoù lamet eus un tu pe eus an daou du";
Blockly.Msg["TODAY"] = "Hiziv";
Blockly.Msg["UNDO"] = "Dizober";
Blockly.Msg["UNKNOWN"] = "Unknown";  // untranslated
Blockly.Msg["UNNAMED_KEY"] = "dizanv";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "elfenn";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Krouiñ 'termenañ %1'";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Distreiñ talvoud an argemm-mañ.";
Blockly.Msg["VARIABLES_SET"] = "termenañ %1 da %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Krouiñ 'kaout %1'";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Termenañ a ra argemm-mañ evit ma vo par da dalvoudenn ar moned.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Un argemmenn anvet '%1' zo anezhi dija.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Un argemenn anvet '%1' zo c'hoazh evit un argemenn all eus ar seurt : '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "A variable named '%1' already exists as a parameter in the procedure '%2'.";  // untranslated
Blockly.Msg["WINDOWS"] = "Windows";  // untranslated
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Lavaret un dra bennak...";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
