// blockly/msg/ce.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "ТӀетоха коммент";
Blockly.Msg["ALT_KEY"] = "Alt";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "ДӀа ца даккхало хийцалург '%1', хӀунда аьлча иза дакъа долун дера функцин билгалонан '%2'";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Хийца маьӀна:";
Blockly.Msg["CHROME_OS"] = "ChromeOS";  // untranslated
Blockly.Msg["CLEAN_UP"] = "ДӀайаха блокаш";
Blockly.Msg["CLOSE"] = "Close";  // untranslated
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "ДӀакъевлина блокашкахь дӀахьедар ду";
Blockly.Msg["COLLAPSE_ALL"] = "Къайлайаха блокаш";
Blockly.Msg["COLLAPSE_BLOCK"] = "Къайлайаккха блок";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "Бос 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "бос 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "бесан дакъа 1";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "иэде";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Ши бос иэбе йеллачу йуъкъаметтигехь (0.0-1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://ce.wikipedia.org/wiki/Бос";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Харжа палитран чуьра бос.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "цахууш нисбелла бос";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Бос ца хууш харжа";
Blockly.Msg["COLOUR_RGB_BLUE"] = "сийна";
Blockly.Msg["COLOUR_RGB_GREEN"] = "баьццара";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "цӀен";
Blockly.Msg["COLOUR_RGB_TITLE"] = "бос оцу чуьра";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Бос бо гайтина пропорцица цӀен, баьццара, сийна. Массо а беснаш 0 а, 100 а йукъахь хила деза.";
Blockly.Msg["COMMAND_KEY"] = "Command";  // untranslated
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "циклера аравалар";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "циклан кхечу гӀулч тӀевала";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "И цикл йукъахйолу";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Йисина цикл чекхйолийтий, дехьайолу кхечу гӀулчан";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "ДӀахьедар: и блок лело мега йеккъа циклан чоьхьахь.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "хӀора элементан %1 испискехь %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Испискера хӀора дакъалган, ло хийцалун '%1' дакъалган маьӀна, кхочушдо гайтина омарш.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "цикл %1 %2 тӀера %3 тӀекхачалц гӀулчца %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Ло хийцалун '%1' маьӀнаш йуьхьнцара чеккхенгара йелла гӀулч, кхочушдо гайтина омарш.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "ТӀетоха блокан биллам";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "ТӀетоха тӀаьххьара бухара блок, массо биллам харц хилча.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "ТӀетоха, дӀайаккха, дехьайаккха блок йухайан дакъалгаш.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "кхин а";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "нагахь кхечу агӀор";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "нагахь";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "МаьӀна нийса мел ду, омарш кхочушдо.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Нагахь биллам нийса балахь, омарийн хьалхара блок кхочушйо. Кхечу агӀора шолгӀа блок кхочушйо.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Нагахь хьалхара биллам бакъ балахь, тӀаккха кхочушйо омарийн хьалхара блок. Кхечу агӀор кхочуш йо омарийн шолгӀа блок.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Нагахь хьалхара биллам нийса балахь, тӀаккха кхочушдо омарийн хьалхара блок.  ШолгӀа биллам нийса балахь кхочушдо омарийн шолгӀа блок. Нагахь цхьан а нийса йацахь, кхочушйо тӀаьххьара омарийн блок.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://en.wikipedia.org/wiki/For_loop";  // untranslated
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "кхочушде";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "%1 йуха арда";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Кхочушдо цхьадолу омарш";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "йухадан, цкъачунна ца оьшу";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "йухаала, цкъачунна";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "МаьӀна харц мел ду, омар кхочушдо.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "МаьӀна нийса мел ду, омарш кхочушдо.";
Blockly.Msg["CONTROL_KEY"] = "Control";  // untranslated
Blockly.Msg["COPY_SHORTCUT"] = "Copy";  // untranslated
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Cut";  // untranslated
Blockly.Msg["DELETE_ALL_BLOCKS"] = "ДӀайаха йерриге блокаш (%1)?";
Blockly.Msg["DELETE_BLOCK"] = "ДӀайаккха блок";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "ДӀайаккха хийцалург %1";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "ДӀадаккха %1 хийцалург лелор %2";
Blockly.Msg["DELETE_X_BLOCKS"] = "ДӀайаха %1 блокаш";
Blockly.Msg["DIALOG_CANCEL"] = "Йухайаккхар";
Blockly.Msg["DIALOG_OK"] = "OK";
Blockly.Msg["DISABLE_BLOCK"] = "Блок дӀайаккха";
Blockly.Msg["DUPLICATE_BLOCK"] = "Копи йан";
Blockly.Msg["DUPLICATE_COMMENT"] = "Комментарийн дубль йар";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Edit Block contents";  // untranslated
Blockly.Msg["ENABLE_BLOCK"] = "Латайе блок";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Гайта блокаш";
Blockly.Msg["EXPAND_BLOCK"] = "Гайта блок";
Blockly.Msg["EXTERNAL_INPUTS"] = "Йллар арахула";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "ГӀо";
Blockly.Msg["HELP_PROMPT"] = "Press %1 for help on keyboard controls.";  // untranslated
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Йллар чухула";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Use the arrow keys to move, then %1 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Copied. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Cut. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Hold %1 and use arrow keys to move freely, then %2 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "йеса исписка кхолла";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Йухайерзайо хаамаш болу йохаллин исписка 0";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "Исписка";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "ТӀетоха, дӀайаккха, дехьайаккха блок йухайан дакъалгаш.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "кхолла могӀам";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "ТӀетоха текстан дакъалг";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Кхуллу исписка муьлххачу дакъалган терахьца.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "хьалхара";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "№ тӀехьара";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "№";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "Схьаэца";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "Схьаэца, дӀадаккха";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "тӀаьххьара";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "ма-луъу";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "дӀайаккха";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Йухайерзайо испискан хьалхара дакъалг";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Йухайерзайо дакъалг испискан гайтинчу меттиге.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Йухайерзайо испискан тӀаьххьара дакъалг.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Йухайерзайо испискан ца хууш нисделла дакъалг.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "ДӀа а йоккху, йуха а йалайо испискан хьалхара дакъалг";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Йухайерзайо дакъалг испискан гайтинчу меттиге.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "ДӀа а йоккху, йуха а йалайо испискан хьалхара дакъалг";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "ДӀа а йоккху, йуха а йалайо испискан ца хууш нисделла дакъалг";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "ДӀадоккху испискан хьалхара дакъалг";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "ДӀадоккху дакъалг испискан гайтинчу меттиге.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "ДӀадоккху испискан тӀаьххьара дакъалг.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "ДӀадоккху испискан ца хууш нисделла дакъалг.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "№ тӀехьара";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "оцу №";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "тӀехьара";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "схьаэца бухара исписка хьалхарчуьнца";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "схьаэца бухара исписка № йолу т\ехьара";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "Схьаэца бухара исписка №";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Гайтина испискин декъера копи кхуллу.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 - тӀаьххьара дакъалг";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 - хьалхара дакъалг";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "Караде хьалхара йозанера чувалар";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "караде тӀаьххьара текстан йуъадар";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Йухайерзайо позицин лоьмар хьалхара/тӀаьххьара могӀам чу элемент йахар. Йухайерзайо %1, нагахь элемент ца карийнехь.";
Blockly.Msg["LISTS_INLIST"] = "испискехь";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 деса ду";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Нагахь исписка йеса йелахь, йухадерзадо бакъдолчун маьӀна";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "дохалла %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Йухайерзайо испискин йохалла";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "кхолла дакъалг %1 исписка, йухайзйеш йолун %2";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Лаьтташ йолу дакъалган копин терахьех исписка кхуллу.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "хийца низам йуханехьа %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Испискан низам йуханехьа хийца";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "=";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "чухӀоттайе";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "дола йаккха";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "ЧухӀоттадо дакъалг испискан йуьххье";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "ЧухӀоттадо дакъалг испискан гайтинчу меттиге.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "ТӀетуху дакъалг испискин чаккхенга";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "ЧухӀоттадо дакъалг ца хууш нисделлачу меттиге испискехь.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Ло испискан хьалхара дакъалг маьӀна.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Ло дакъалг испискан гайтинчу меттиге.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Ло маьӀна испискан тӀаьххьара дакъалг.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Ло испискан ца хууш нисделла дакъалган маьӀна.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "йоккха хиларца";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "кӀезиг хиларца";
Blockly.Msg["LISTS_SORT_TITLE"] = "%1 %2 %3 сорт хӀоттайо";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Испискан копин сорт хӀоттор";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "Абатца, регистран чот а ца лелош";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "лоьмаран";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "абатца";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "йозанах исписка йар";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "текст вовшахтоха могӀанах";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Вовшахтуху текстийн могӀам цхьаьна тексте доькъучуьнца.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Йоькъу текст могӀаман текст тӀе доькъучуьнца";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "доькъучунца";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "харц дерг";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Йухадерзадо бакъ йа харц долу маьӀна";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "бакъдерг";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://en.wikipedia.org/wiki/Inequality_(mathematics)";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Нагахь чудаьхнарш тера делахь, йухадогӀу аьтту маьӀна.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Нагахь хьалхара диллар шолгӀачул дукха делахь, йухадогӀу бакъ долу маьӀна";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Хьалха чуйаьккхинарг кӀезиг йалахь шолгӀачул йа и санна йалахь, йухадогӀу бакъ долу маӀна.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Нагахь хьалха чудаьккхинарг кӀезиг делахь шолгӀачул, йухадогӀу аьтту маьӀна.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Хьалха чуйаьккхинарг кӀезиг йалахь шолгӀачул йа и санна йалахь, йухадогӀу бакъ долу маӀна.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Нагахь чудаьхнарш тера дацахь, йухадогӀу аьтту болу маьӀнаш.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "хӀанхӀа %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Нагахь диллар харц далахь, йухадерзадо бакъ долу маьӀна. Нагахь диллар нийса делахь, йухадерзадо харц маьӀна.";
Blockly.Msg["LOGIC_NULL"] = "ХӀумма а доцург";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Йухайерзайо хӀумма а йоцург";
Blockly.Msg["LOGIC_OPERATION_AND"] = "а,";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "йа";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Нагахь ший а диллар нийса делахь, йухадогӀу бакъ долу маьӀна.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Цхьа диллар мукъане а нийса делахь, йухадаладо бакъ долу маьӀна.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "харжа";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "нагахь харц делахь";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "нагахь бакъ делахь";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Толлу харжаман хьал. Нагахь хьал бакъ делахь, йухадерзадо хьалхара маьӀна, кхечу агӀор йухадерзадо шолгӀа маьӀна.";
Blockly.Msg["MAC_OS"] = "macOS";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://en.wikipedia.org/wiki/Arithmetic";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "йухайерзор ду ши терахь вовшахтоьхна лоьмар";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Йухадерзор ду доькъург доькъучу хьалхара терхьах шолгӀачун";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "йухадерзор ду тӀера дӀадаьккхина ши терахь";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Йухадерзор ду шозза эцна терахь";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Йухадерзадо хьалхар терахь, айина долу шолгӀачу терхьан дарже.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 от X:%1 Y:%2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Йухайерхайо меттиг (X, Y) арктангенс градусашкахь -180 тӀера 180 кхаччалц.";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "доккха дан %1 тӀера %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "ТӀетоха хийцалун терахь %1";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://en.wikipedia.org/wiki/Mathematical_constant";  // untranslated
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Йухайерзайо уггаре йаьржина констатех цхьаъ: π (3.141...), e (2.718...), φ (1.618...), sqrt(2) (1.414...), sqrt(½) (0.707...), ∞ (чеккхейоцург).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "дозатуху %1 лахахь %2 лакхахь %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Дозатуху лахара а, лакхара а дозанца.";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "декъало";
Blockly.Msg["MATH_IS_EVEN"] = "Чоьтан";
Blockly.Msg["MATH_IS_NEGATIVE"] = "аьтту боцуш";
Blockly.Msg["MATH_IS_ODD"] = "Чоьтан йоцу";
Blockly.Msg["MATH_IS_POSITIVE"] = "аьтту болуш";
Blockly.Msg["MATH_IS_PRIME"] = "атта";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Толлу, дуй теша терахь шала, шалза, атта, дийна, аьтту, аьттунза йа иза декъало билгалчу терахьан. Йухадерзадо маьӀна бакъ дерг йа харц дерг.";
Blockly.Msg["MATH_IS_WHOLE"] = "дийна";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://en.wikipedia.org/wiki/Modulo_operation";  // untranslated
Blockly.Msg["MATH_MODULO_TITLE"] = "Йисина %1 тӀера %2 кхаччалц";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Ши терахь вовшенна декъча бухадисинарг.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://en.wikipedia.org/wiki/Number";  // untranslated
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Лоьмар";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "Йуккъера арифметикан исписка";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "испискера уггаре дукханиг";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "испискан медиана";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "испискехь уггаре кӀезганиг";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "испискан моданаш";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "испискан цахууш хилла дакъалг";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "испискехь стандартан дӀахилар";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "исписка вовшахтоьхна";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Испискера йуккъера арифметикан йухадерзадо.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Исписмкера уггаре дукха терахь йухадерзадо";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Испискан медиана йухадерзадо";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Йузадерзадо уггаре кӀезга терахь испискера";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Испискан дукха хаалуш долу дакъалг исписке йухадерзадо.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Испискехь цахууш хилла дакъалг йухадерзадо";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Йухадерзадо испискан стандартан дӀахилар";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Испискера терахьийн жамӀ йухадерзадо";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "0.0 тӀера 1.0 кхаччалц (йукъахь долуш) цӀеххана нисделла дийна терахь";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Йухедерзадо цӀеххьана нисделла дийна терахь 0.0 тӀера 1.0 кхаччалц";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "цӀеххьана нисделла дийна терахь %1 тӀера %2 кхаччалц";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Йухадерзадо цӀеххьана нисделла шина дозанна йукъара.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "горгдан";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "Ден дан лаха";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "Горгдан лакха";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Горгдан лакха йа лаха";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://en.wikipedia.org/wiki/Square_root";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "дуьззина";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "БеакӀов орам";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Йухадерзадо дуьззина терахь";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Йухайерзайо гайтинчу даржехь.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Йухадерзадо терхьан бакъ йолу логарифм.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Йухайерзайо терхьан иттаннийн логарифм";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Йухадерзадо дуьхь-дуьхьал долу терхьаш";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Йухайерзайо 10 гайтинчу даржехь";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Йухаберзабо терхьан беакӀов орам";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "акос";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "муха";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "вагавар";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "хӀунда аьлча";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://en.wikipedia.org/wiki/Trigonometric_functions";  // untranslated
Blockly.Msg["MATH_TRIG_SIN"] = "къа";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "вагавар";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Йухайерзайе терхьан арккосинус (градусашкахь)";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Йухайерзайо арксинус (градусашкахь).";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Йухайерзайе арктангенс (градусашкахь)";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Йухайерзайо сенан косинус градусашкахь.";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Йухайерзайо сенан маьӀиг градусашкахь";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Йухайерзайо тангенсан сенан маьӀиг градусашкахь";
Blockly.Msg["MOVE_BLOCK"] = "Move Block";  // untranslated
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Бесан хийцалург йе";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Терхьан хийцалург кхолла";
Blockly.Msg["NEW_STRING_VARIABLE"] = "МогӀанна хийцалург йе...";
Blockly.Msg["NEW_VARIABLE"] = "Хийцалун цӀе кхолла...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Керла хийцалун цӀе:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Керла хийцалун цӀе:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "Option";  // untranslated
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Paste";  // untranslated
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "операторшан пурба дала";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "дуьйна:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Кхочушйо лелочуьнан билгала процедура '%1'";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Кхочушйо лелочуьнан билгала процедура '%1', йухадерзадо лерина маьӀна";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "дуьйна:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Декъашхочун '%1' функци дӀайахьа йиш йац, хӀунда аьлча билгалйаьккхина блок дӀайаьккхина йу.";
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Кхолла %1 кхайкхам";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Гайта и функци";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "кхочушдан цхьа хӀума";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "хӀунда аьлча";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Кхуллу маьӀна йуха ца дерзош йолу процедура";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "йухадерзо";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Кхуллу маьӀна йуха дерзош йолу процедура";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "ДӀахьедар: оцу функцин йух-йуха тӀейуйлу параметраш йу.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Къастайе процедурин билгало";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Нагахь хьалхара маьӀна нийса делалахь, йухадерзадо шолгӀа маьӀна.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "ДӀахедар: ХӀокху блоко лелон таро йу функцин чоьхьара билгало.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "параметран цӀе";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Чуволу параметран функци тӀетоха";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "параметарш";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "ТӀетоха, дӀадаккха йа хийца оцу чуйогӀу параметрийн функцеш.";
Blockly.Msg["REDO"] = "Йухайерзор";
Blockly.Msg["REMOVE_COMMENT"] = "ДӀайаккха коммент";
Blockly.Msg["RENAME_VARIABLE"] = "Хийцалун цӀе хийца...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Массо хийцалун цӀерш хийца '%1' тӀе:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Code navigation";  // untranslated
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Editing";  // untranslated
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "General";  // untranslated
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "%1 тӀетоха текст %2";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "ТӀетоха текст хийцалучунна \"%1\"";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "могӀанан элпаш";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "Хьалхарчу Коьрта Элпашка";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "ХЬАЛХАРЧУ ЭЛПАШКА";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Йухайерзайо текстан копи ХЬАЛХАРЧУ йа могӀанан элпашца";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "хьалхара элп схьаэца";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "№ йолу элп схьаэца тӀехььара";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "№ йолу элп схьаэца";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "тӀаьххьара элп схьаэца";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "Схаэца ца хоржуш элп";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "Йозанехь %1, %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Йухадерзадо элп гайтинчу меттиге";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "Дагарде барам %1 %2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Дагарде, мосазза йолу текстан кийсиг кхечу текстехь гучуйолу.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "ТӀетоха текстан дакъалг";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "вовшахтаса";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "ТӀетоха, дӀайаккха, дехьайаккха блок йухайан дакъалгаш.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "№ йолу элп схьаэца тӀехьара";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "№ йолу элп схьаэца";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "тӀаьххьара элп схьаэца";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "Йозанехь";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "схьаэца бухара могӀа хьалхарчу элпера";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "Схьаэца бухара могӀа № йолу элпера тӀехьара";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "Схьаэца бухара могӀа № йолу элпера";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Текстан гайтина дакъа йухадерзадо";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "Караде хьалхара йозанера чувалар";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "караде тӀаьххьара текстан йукъайар";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "Йозанехь %1, %2, %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Йухайерзайо позицин лоьмар хьалхара/тӀаьххьара хьалхара текст шолгӀачун йукъа йахар. Йухайерзайо %1, нагахь текст ца карийнехь.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 деса ду";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Нагахь текст йаьсса йелахь, йухадерзадо бакъ долу маьӀна.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "кхолла могӀам";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Вовшахтосу муьлха дакъалган терахь, кхуллу текстан кийсиг";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "дохалла %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Йухадерзадо сийлаллин терахь деллачу йозанехь";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "Зорбатоха %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Зорбатуху текстан, терахьан йа кхечу объектан";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Лелочунга терахь деха.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Текст йеха декъашхочунга.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "терахь деха дӀааларца";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "текст деха дӀааларца";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "хийца %1 %2 %3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Хийца ша долу дерриг йозананна чуваларш, кхечу йозанца";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "хийца низам йуханехьа %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Хуьйцу сийлаллин низам йуханехьа долчуьнца.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://en.wikipedia.org/wiki/String_(computer_science)";  // untranslated
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Элп, дош, могӀам йозанехь.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "Йаккъаш дӀахедайар";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "аьрру агӀор йукъ дӀахадайар";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "Аьтту агӀор йаккъаш дӀахедайе";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Йухайерзайо текстан копи генайаьккхина йукъ цхьаьна йа шинне а йуьххьера";
Blockly.Msg["TODAY"] = "Тахана";
Blockly.Msg["UNDO"] = "Йухадаккха";
Blockly.Msg["UNKNOWN"] = "Unknown";  // untranslated
Blockly.Msg["UNNAMED_KEY"] = "цӀе йац";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "элемент";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Блок кхолла \"ло\" %1";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Оцу хийцалучун маьӀна хуьйцу";
Blockly.Msg["VARIABLES_SET"] = "ло %1=%2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Кхолла чухӀотториг %1";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Ло хийцалучн чухӀотточун маьӀна.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Хийцалург %1 цӀарца хӀинцале йолуш йу";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Хийцалучун цӀе '%1' хӀинцале лелош кхечу тайпано: '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "Хийцалуг цӀарца '%1' хӀинцале йолуш ю процедурехь параметр '%2' хьокъехь.";
Blockly.Msg["WINDOWS"] = "Windows";  // untranslated
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Йазйе кхузахь цхьа хӀума...";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
