// blockly/msg/cs.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Přidat komentář";
Blockly.Msg["ALT_KEY"] = "Alt";
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Proměnnou '%1' není možné odstranit, protože je součástí definice funkce '%2'";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Změnit hodnotu:";
Blockly.Msg["CHROME_OS"] = "ChromeOS";
Blockly.Msg["CLEAN_UP"] = "Uspořádat bloky";
Blockly.Msg["CLOSE"] = "Zavřít";
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Sbalené bloky obsahují varování.";
Blockly.Msg["COLLAPSE_ALL"] = "Sbalit bloky";
Blockly.Msg["COLLAPSE_BLOCK"] = "Sbalit blok";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "barva 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "barva 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "poměr";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "smíchat";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Smíchá dvě barvy v daném poměru (0.0–1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://cs.wikipedia.org/wiki/Barva";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Vyberte barvu z palety.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "náhodná barva";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Zvolte barvu náhodně.";
Blockly.Msg["COLOUR_RGB_BLUE"] = "modrá";
Blockly.Msg["COLOUR_RGB_GREEN"] = "zelená";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "červená";
Blockly.Msg["COLOUR_RGB_TITLE"] = "obarvěte barvou";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Vytvoř barvu se zadaným množstvím červené, zelené a modré.  Všechny hodnoty musí být mezi 0 a 100.";
Blockly.Msg["COMMAND_KEY"] = "⌘ Příkaz";
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "vyskočit ze smyčky";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "pokračuj dalším opakováním smyčky";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Vyskoč z vnitřní smyčky.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Přeskoč zbytek této smyčky a pokračuj dalším opakováním.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Upozornění: Tento blok může být použit pouze uvnitř smyčky.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "pro každou položku %1 v seznamu %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Pro každou položku v seznamu nastavte do proměnné '%1' danou položku a proveďte nějaké operace.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "počítat s %1 od %2 do %3 po %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Nechá proměnnou '%1' nabývat hodnot od počátečního do koncového čísla po daném přírůstku a provádí příslušné bloky.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Přidat podmínku do \"pokud\" bloku.";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Přidej konečnou podmínku zahrnující ostatní případy do bloku \"pokud\".";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Přidej, odstraň či uspořádej sekce k přenastavení tohoto bloku \"pokud\".";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "jinak";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "nebo pokud";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "pokud";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Je-li hodnota pravda, proveď určité příkazy.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Je-li hodnota pravda, proveď první blok příkazů.  V opačném případě proveď druhý blok příkazů.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Je-li první hodnota pravdivá, proveď první blok příkazů. V opačném případě, je-li pravdivá druhá hodnota, proveď druhý blok příkazů.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Je-li první hodnota pravda, proveď první blok příkazů.  Je-li druhá hodnota pravda, proveď druhý blok příkazů.  Pokud žádná hodnota není pravda, proveď poslední blok příkazů.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://cs.wikipedia.org/wiki/Cyklus_for";
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "dělej";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "opakuj %1 krát";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Proveď určité příkazy několikrát.";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "opakovat dokud";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "opakovat když";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Dokud je hodnota nepravdivá, prováděj určité příkazy.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Dokud je hodnota pravdivá, prováděj určité příkazy.";
Blockly.Msg["CONTROL_KEY"] = "Ctrl";
Blockly.Msg["COPY_SHORTCUT"] = "Kopírovat";
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Vyjmout";
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Smazat všech %1 bloků?";
Blockly.Msg["DELETE_BLOCK"] = "Smazat blok";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Odstranit proměnnou '%1'";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Odstranit %1 použití proměnné '%2'";
Blockly.Msg["DELETE_X_BLOCKS"] = "Smazat %1 bloků";
Blockly.Msg["DIALOG_CANCEL"] = "Zrušit";
Blockly.Msg["DIALOG_OK"] = "OK";
Blockly.Msg["DISABLE_BLOCK"] = "Deaktivovat blok";
Blockly.Msg["DUPLICATE_BLOCK"] = "Duplikovat";
Blockly.Msg["DUPLICATE_COMMENT"] = "Duplikovat komentář";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Edit Block contents";  // untranslated
Blockly.Msg["ENABLE_BLOCK"] = "Povolit blok";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Rozbalit bloky";
Blockly.Msg["EXPAND_BLOCK"] = "Rozbalit blok";
Blockly.Msg["EXTERNAL_INPUTS"] = "Vnější vstupy";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Nápověda";
Blockly.Msg["HELP_PROMPT"] = "Stiskněte %1 pro nápovědu k ovládání pomocí klávesnice";
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Vložené vstupy";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Použijte šipky k pohybu, poté stiskněte %1 pro potvrzení pozice";
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Zkopírováno. Stiskněte %1 pro vložení.";
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Vyjmuto. Stiskněte %1 pro vložení.";
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Podržte %1 k volnému pohybu pomocí šipek, poté pomocí %2 pozici potvrďte";
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "vytvořit prázdný seznam";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Vrátí seznam nulové délky, který neobsahuje žádné datové záznamy";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "seznam";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Přidat, odebrat nebo změnit pořadí oddílů tohoto seznamu bloku.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "vytvořit seznam s";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Přidat položku do seznamu.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Vytvoř seznam s libovolným počtem položek.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "první";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# od konce";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "získat";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "získat a odstranit";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "poslední";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "náhodné";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "odstranit";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Vrátí první položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Získá položku z určené pozice v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Vrátí poslední položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Vrátí náhodnou položku ze seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Odstraní a vrátí první položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Odstraní a získá položku z určené pozice v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Odstraní a vrátí poslední položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Odstraní a vrátí náhodnou položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Odstraní první položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Odebere položku na konkrétním místě v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Odstraní poslední položku v seznamu.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Odstraní náhodou položku v seznamu.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "do # od konce";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "do #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "jako poslední";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "získat podseznam od první položky";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "získat podseznam od # od konce";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "získat podseznam od #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Vytvoří kopii určené části seznamu.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 je poslední položka.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 je první položka.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "najít první výskyt položky";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "najít poslední výskyt položky";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Vrací index prvního/posledního výskytu položky v seznamu. Vrací %1, pokud položka nebyla nalezena.";
Blockly.Msg["LISTS_INLIST"] = "v seznamu";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 je prázdné";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Vrátí hodnotu pravda, pokud je seznam prázdný.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "délka %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Vrací počet položek v seznamu.";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "vytvoř seznam s položkou %1 opakovanou %2 krát";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Vytváří seznam obsahující danou hodnotu n-krát.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "obrátit %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Obrátit kopii seznamu.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "jako";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "vložit na";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "nastavit";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Vložit položku na začátek seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Vloží položku na určenou pozici v seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Připojí položku na konec seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Připojí položku náhodně do seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Nastaví první položku v seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Nastaví položku na konkrétní místo v seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Nastaví poslední položku v seznamu.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Nastaví náhodnou položku v seznamu.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "vzestupně";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "sestupně";
Blockly.Msg["LISTS_SORT_TITLE"] = "seřadit %1 %2 %3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Seřadit kopii seznamu.";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "abecedně, na velikosti písmen nezáleží";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "číselné";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "abecedně";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "udělat z textu seznam";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "udělat ze seznamu text";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Spojit seznam textů do jednoho textu, rozdělaného oddělovači.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Rozdělit text do seznamu textů, lámání na oddělovačích.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "s oddělovačem";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "nepravda";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Vrací pravda nebo nepravda.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "pravda";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://cs.wikipedia.org/wiki/Nerovnost_(matematika)";
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Vrátí hodnotu pravda, pokud se oba vstupy rovnají jeden druhému.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Navrátí hodnotu pravda, pokud první vstup je větší než druhý vstup.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Navrátí hodnotu pravda, pokud je první vstup větší a nebo rovný druhému vstupu.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Navrátí hodnotu pravda, pokud je první vstup menší než druhý vstup.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Navrátí hodnotu pravda, pokud je první vstup menší a nebo rovný druhému vstupu.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Vrátí hodnotu pravda, pokud se oba vstupy nerovnají sobě navzájem.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "ne %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Navrátí hodnotu pravda, pokud je vstup nepravda. Navrátí hodnotu nepravda, pokud je vstup pravda.";
Blockly.Msg["LOGIC_NULL"] = "prázdný";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Vrátí prázdnou hodnotu";
Blockly.Msg["LOGIC_OPERATION_AND"] = "a";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "nebo";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Vrátí hodnotu pravda, pokud oba dva vstupy jsou pravdivé.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Vrátí hodnotu pravda, pokud alespoň jeden ze vstupů má hodnotu pravda.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "test";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://cs.wikipedia.org/wiki/Ternární operátor (programování)";
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "pokud nepravda";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "pokud pravda";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Zkontroluje podmínku v \"testu\". Když je podmínka pravda, vrátí hodnotu \"pokud pravda\"; v opačném případě vrátí hodnotu \"pokud nepravda\".";
Blockly.Msg["MAC_OS"] = "macOS";
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://cs.wikipedia.org/wiki/Aritmetika";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Vrátí součet dvou čísel.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Vrátí podíl dvou čísel.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Vrátí rozdíl dvou čísel.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Vrátí součin dvou čísel.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Vrátí první číslo umocněné na druhé číslo.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://cs.wikipedia.org/wiki/Atan2";
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 z X:%1 Y:%2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Vrací arkustangens bodu (X, Y) ve stupních od -180 do 180.";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "zaměň %1 za %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Přičti číslo k proměnné '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://en.wikipedia.org/wiki/Mathematical_constant";  // untranslated
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Vraťte jednu z následujících konstant: π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), or ∞ (nekonečno).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "omez %1 na rozmezí od %2 do %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Omezí číslo tak, aby bylo ve stanovených mezích (včetně).";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "je dělitelné číslem";
Blockly.Msg["MATH_IS_EVEN"] = "je sudé";
Blockly.Msg["MATH_IS_NEGATIVE"] = "je záporné";
Blockly.Msg["MATH_IS_ODD"] = "je liché";
Blockly.Msg["MATH_IS_POSITIVE"] = "je kladné";
Blockly.Msg["MATH_IS_PRIME"] = "je prvočíslo";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Kontrola, zda je číslo sudé, liché, prvočíslo, celé, kladné, záporné nebo zda je dělitelné daným číslem. Vrací pravdu nebo nepravdu.";
Blockly.Msg["MATH_IS_WHOLE"] = "je celé";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://cs.wikipedia.org/wiki/Modul%C3%A1rn%C3%AD_aritmetika";
Blockly.Msg["MATH_MODULO_TITLE"] = "zbytek po dělení %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Vrátí zbytek po dělení dvou čísel.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://cs.wikipedia.org/wiki/Číslo";
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Číslo.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "průměr v seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "největší v seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "medián v seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "nejmenší v seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "nejčastější ze seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "náhodná položka seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "směrodatná odchylka ze seznamu";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "suma seznamu";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Vrátí průměr (aritmetický průměr) číselných hodnot v seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Vrátí největší číslo v seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Vrátí medián seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Vrátí nejmenší číslo v seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Vrátí seznam nejčastějších položek seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Vrátí náhodnou položku ze seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Vrátí směrodatnou odchylku seznamu.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Vrátí součet všech čísel v seznamu.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://cs.wikipedia.org/wiki/Gener%C3%A1tor_n%C3%A1hodn%C3%BDch_%C4%8D%C3%ADsel";
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "náhodné číslo mezi 0 (včetně) do 1";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Vrátí náhodné číslo mezi 0,0 (včetně) až 1,0";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://cs.wikipedia.org/wiki/Gener%C3%A1tor_n%C3%A1hodn%C3%BDch_%C4%8D%C3%ADsel";
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "náhodné celé číslo od %1 do %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Vrací náhodné celé číslo mezi dvěma určenými mezemi, včetně mezních hodnot.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "zaokrouhlit";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "zaokrouhlit dolů";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "zaokrouhlit nahoru";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Zaokrouhlit číslo nahoru nebo dolů.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://cs.wikipedia.org/wiki/Druhá_odmocnina";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "absolutní hodnota";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "druhá odmocnina";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Vrátí absolutní hodnotu čísla.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Vrátí mocninu čísla e.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Vrátí přirozený logaritmus čísla.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Vrátí desítkový logaritmus čísla.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Vrátí zápornou hodnotu čísla.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Vrátí mocninu čísla 10.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Vrátí druhou odmocninu čísla.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "arcsin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "arctan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "cos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://en.wikipedia.org/wiki/Trigonometric_functions";  // untranslated
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Vrátí arkus kosinus čísla.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Vrátí arkus sinus čísla.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Vrátí arkus tangens čísla.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Vrátí kosinus úhlu ve stupních.";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Vrátí sinus úhlu ve stupních.";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Vrátí tangens úhlu ve stupních.";
Blockly.Msg["MOVE_BLOCK"] = "Přesunout blok";
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Vytvořit proměnnou pro barvu...";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Vytvořit číselnou proměnnou...";
Blockly.Msg["NEW_STRING_VARIABLE"] = "Vytvořit textovou proměnnou...";
Blockly.Msg["NEW_VARIABLE"] = "Vytvořit proměnnou...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Nový název proměnné:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Nový typ proměnné:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "⌥ Možnost";
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Vložit";
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "povolit příkazy";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "s:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://cs.wikipedia.org/wiki/Podprogram";
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Spustí uživatelem definovanou funkci '%1'.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://cs.wikipedia.org/wiki/Podprogram";
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Spustí uživatelem definovanou funkci '%1' a použije její výstup.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "s:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Can't run the user-defined function '%1' because the definition block is disabled.";  // untranslated
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Vytvořit '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Popište tuto funkci...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "proveď něco";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "k provedení";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Vytvořit funkci bez výstupu.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://cs.wikipedia.org/wiki/Podprogram";
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "navrátit";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Vytvořit funkci s výstupem.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Upozornění: Tato funkce má duplicitní parametry.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Zvýraznit definici funkce";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Je-li hodnota pravda, pak vrátí druhou hodnotu.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Varování: Tento blok může být použit pouze uvnitř definici funkce.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "vstupní jméno:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Přidat vstupy do funkce.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "vstupy";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Přidat, odebrat nebo změnit pořadí vstupů této funkce.";
Blockly.Msg["REDO"] = "Znovu";
Blockly.Msg["REMOVE_COMMENT"] = "Odstranit komentář";
Blockly.Msg["RENAME_VARIABLE"] = "Přejmenovat proměnnou...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Přejmenuj všech '%1' proměnných na:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Code navigation";  // untranslated
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Editace";
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "Obecné";
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "do %1 přidat text %2";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Přidá určitý text k proměnné '%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "na malá písmena";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "na Počáteční Velká Písmena";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "na VELKÁ PÍSMENA";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Vrátí kopii textu s jinou velikostí písmen.";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "získat první písmeno";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "získat # písmeno od konce";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "získat písmeno #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "získat poslední písmeno";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "získat náhodné písmeno";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "v textu %1 %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Získat písmeno na konkrétní pozici.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "počet výskytů %1 v %2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Spočítat, kolikrát se daný text vyskytuje v jiném textu.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Přidat položku do textu.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "spojit";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Přidat, odebrat nebo změnit pořadí oddílů tohoto textového bloku.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "do # písmene od konce";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "do písmene #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "do posledního písmene";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "v textu";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "získat podřetězec od prvního písmene";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "získat podřetězec od písmene # od konce";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "získat podřetězec od písmene #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Získat zadanou část textu.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "najít první výskyt textu";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "najít poslední výskyt textu";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "v textu %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Vrátí index prvního/posledního výskytu prvního textu v druhém textu.  Pokud text není nalezen, vypíše %1.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 je prázdný";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Vrátí pravda pokud je zadaný text prázdný.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "vytvořit text s";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Vytvoří kousek textu spojením libovolného počtu položek.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "délka %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Vrátí počet písmen (včetně mezer) v zadaném textu.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "tisk %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Tisk zadaného textu, čísla nebo jiné hodnoty.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Výzva pro uživatele k zadání čísla.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Výzva pro uživatele k zadání nějakého textu.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "výzva k zadání čísla se zprávou";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "výzva k zadání textu se zprávou";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "nahradit %1 za %2 v řetězci %3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Nahradit všechny výskyty daného textu v jiném textu.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "obrátit %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Obrátit pořadí znaků v textu.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://cs.wikipedia.org/wiki/Textov%C3%BD_%C5%99et%C4%9Bzec";
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Písmeno, slovo nebo řádek textu.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "odstranit mezery z obou stran";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "odstranit mezery z levé strany";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "odstranit mezery z pravé strany";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Vrátí kopii textu s odstraněnými mezerami z jednoho nebo obou konců.";
Blockly.Msg["TODAY"] = "Dnes";
Blockly.Msg["UNDO"] = "Zpět";
Blockly.Msg["UNKNOWN"] = "Neznámý";
Blockly.Msg["UNNAMED_KEY"] = "nepojmenovaný";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "položka";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Vytvořit \"nastavit %1\"";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Vrátí hodnotu této proměnné.";
Blockly.Msg["VARIABLES_SET"] = "nastavit %1 na %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Vytvořit \"získat %1\"";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Nastaví tuto proměnnou, aby se rovnala vstupu.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Proměnná jménem '%1' již existuje";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Proměnná pojmenovaná jako '%1' již existuje pro jiný typ: '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "Proměnná pojmenovaná „%1“ již jako parametr v proceduře „%2“ existuje.";
Blockly.Msg["WINDOWS"] = "Windows";
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Say something...";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
