// blockly/msg/ga.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Cuir ráiteas";
Blockly.Msg["ALT_KEY"] = "Alt";
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Ní féidir an athróg %1 a scrios mar tá sé mar phairt den chuspóir %2";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Athraigh luach";
Blockly.Msg["CHROME_OS"] = "ChromeOS";
Blockly.Msg["CLEAN_UP"] = "Glan suas na blic";
Blockly.Msg["CLOSE"] = "Dún";
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Tá rabhaidh i mblocanna fillte.";
Blockly.Msg["COLLAPSE_ALL"] = "Druid síos na blic";
Blockly.Msg["COLLAPSE_BLOCK"] = "Druid síos an bloc";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "dath 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "dath 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "cóimheas";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "measc";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Meascann an dá dhath le chéile le cóimheas tughta (0.0-1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://en.wikipedia.org/wiki/Color";  // untranslated
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Roghnaigh do dhath ón bpailéad.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "Dath randamach";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Roghnaigh dath go randamach";
Blockly.Msg["COLOUR_RGB_BLUE"] = "gorm";
Blockly.Msg["COLOUR_RGB_GREEN"] = "glas";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "dearg";
Blockly.Msg["COLOUR_RGB_TITLE"] = "Dathaigh le";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "measc";
Blockly.Msg["COMMAND_KEY"] = "⌘ Command";
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "bris amach as lúb";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "lean ar aghaidh leis an athuair eile den lúb";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Bris amach as an choinneail lúb.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Léim thar an chuid eile den lúb seo, agus lean ar aghaidh leis an chéad athuair eile.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Rabhadh: Níl cead an bloc seo a úsáid ach amháin lastigh de lúb.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "do gach bhall %1 i liosta %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Do gach mír i liosta, socraigh an t-athróg %1 go dtí an mír, agus ansin déan grain ráitéisí.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "cuntas le %1 ó %2 go %3 ua %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Faigh ar an t-athróg %1 na luachanna a thógáil ón chéad uimhir go dtí an uimhir deiridh, ag comhaireamh leis an eatramh sonraithe,agus déan na bloic sonraithe.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Cuir coinníoll leis an mbloc \"if\".";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Cuir coinníoll deiridh, uileghabhálach leis an mbloc \"if\".";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Cuir rannóga leis, bain iad, nó athordaigh iad chun an bloc seo a athchumrú.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "eile";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "eile má";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "Má";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Má tá an luach fíór, déan grain raitéis.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Más fíor luach, déan an chéad bhloc ráiteas. Seachas sin, déan an dara bloc ráiteas.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Más fíor an chéad luach, déan an chéad bhloc ráiteas. Seachas sin, más fíor an dara luach, déan an dara bloc ráiteas.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Más fíor an chéad luach, déan an chéad bhloc ráiteas. Seachas sin, más fíor an dara luach, déan an dara bloc ráiteas. Mura bhfuil aon cheann de na luachanna fíor, déan an bloc ráiteas deireanach.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://en.wikipedia.org/wiki/For_loop";  // untranslated
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "déan";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "Athdhéan %1 uairenata";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Déan cúpla ráiteas cúpla uair.";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "arís go dtí";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "arís fhad 's atá";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Fhad is atá luach bréagach, ansin déan grain ráitisí.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Fhad is atá luach fíór, ansin déan grain ráitisí.";
Blockly.Msg["CONTROL_KEY"] = "Ctrl";
Blockly.Msg["COPY_SHORTCUT"] = "Cóipeáil";
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Gearr";
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Scrios  %1 bloc uilig?";
Blockly.Msg["DELETE_BLOCK"] = "Scrios Bloc";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Scrios an t-athróg %1";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Scrios %1 úsádí den athróg %2";
Blockly.Msg["DELETE_X_BLOCKS"] = "Scrios %1 Blic";
Blockly.Msg["DIALOG_CANCEL"] = "Cealaigh";
Blockly.Msg["DIALOG_OK"] = "Ceart go leor";
Blockly.Msg["DISABLE_BLOCK"] = "Díchumasaigh Bloc";
Blockly.Msg["DUPLICATE_BLOCK"] = "Dúblach";
Blockly.Msg["DUPLICATE_COMMENT"] = "Dúblach an ráiteas";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Cuir ábhar an bhloic in eagar";
Blockly.Msg["ENABLE_BLOCK"] = "Cumasaigh Bloc";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Leathnaigh Blic";
Blockly.Msg["EXPAND_BLOCK"] = "Leathnaigh bloc";
Blockly.Msg["EXTERNAL_INPUTS"] = "Cuir isteach Seachtrach";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Cabhrú";
Blockly.Msg["HELP_PROMPT"] = "Brúigh %1 le haghaidh cabhrach ar rialuithe méarchláir";
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Cuir isteach inlíne";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Úsáid na heochracha saigheada chun bogadh, ansin %1 chun glacadh leis an suíomh";
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Cóipeáilte. Brúigh %1 le greamaigh.";
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Gearr. Brúigh %1 le greamaigh.";
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Coinnigh %1 agus bain úsáid as na heochracha saigheada chun bogadh go saor, ansin %2 chun glacadh leis an suíomh";
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "liosta folamh a chruthú";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Seolann sé liosta ar ais, de fhad 0, nach bhfuil aon taifid sonraí ann";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "liosta";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Cuir rannóga leis, bain iad, nó athordaigh iad chun an bloc liosta seo a athchumrú.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "cruthaigh liosta le";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Cuir mír leis an liosta.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Cruthaigh liosta le haon líon míreanna.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "ar dtús";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# ón deireadh";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "faigh";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "faigh agus bain";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "deireanach";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "randamach";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "bain";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Tugann sé an chéad mhír i liosta ar ais.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Tugann sé an mhír ar ais ag an suíomh sonraithe i liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Tugann sé an mhír dheireanach i liosta ar ais.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Tugann sé mír randamach ar ais i liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Baintear an chéad mhír i liosta agus cuirtear ar ais í.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Baintear an mhír agus cuirtear ar ais í ag an suíomh sonraithe i liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Baintear an mhír dheireanach i liosta agus tugann sí ar ais í.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Baintear mír randamach as liosta agus tugann sé ar ais í.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Baintear an chéad mhír as liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Baintear an mhír ag an suíomh sonraithe i liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Baintear an mhír dheireanach i liosta.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Baintear mír randamach as liosta.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "go # ón deireadh";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "go #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "chun deireanach";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "faigh fo-liosta ón gcéad cheann";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "faigh fo-liosta ó # ón deireadh";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "faigh fo-liosta ó #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Cruthaíonn sé cóip den chuid shonraithe de liosta.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "Is é %1 an mhír dheireanach.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "Is é %1 an chéad mhír.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "aimsigh an chéad uair a tharlaíonn mír";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "faigh an uair dheireanach den mhír";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Seolann sé innéacs an chéad/an uair dheireanaigh den mhír sa liosta ar ais. Seolann sé %1 ar ais mura bhfuarthas an mhír.";
Blockly.Msg["LISTS_INLIST"] = "sa liosta";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "Tá %1 folamh";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Tugann sé fíor ar ais má tá an liosta folamh.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "fad %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Tugann sé fad liosta ar ais.";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "cruthaigh liosta le mír %1 arís %2 uair";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Cruthaíonn sé liosta ina bhfuil an luach tugtha arís agus arís eile an líon sonraithe uaireanta.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "droim ar ais %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Cóip de liosta a aisiompú.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "mar";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "cuir isteach ag";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "socraigh";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Cuireann sé an mhír ag tús liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Cuireann sé an mhír isteach san áit shonraithe i liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Cuir an mhír le deireadh liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Cuireann sé an mhír isteach go randamach i liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Socraíonn sé an chéad mhír i liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Socraíonn sé an mhír ag an suíomh sonraithe i liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Socraíonn sé an mhír dheireanach i liosta.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Socraíonn sé mír randamach i liosta.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "ag dul suas";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "ag dul síos";
Blockly.Msg["LISTS_SORT_TITLE"] = "sórtáil %1 %2 %3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Sórtáil cóip de liosta.";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "aibítreach, neamhaird a thabhairt ar chás";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "uimhriúil";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "aibítreach";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "déan liosta ón téacs";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "déan téacs ón liosta";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Ceangail liosta téacsanna in aon téacs amháin, scartha le teormharcóir.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Scoilt téacs ina liosta téacsanna, ag briseadh ag gach deighilteoir.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "le deighilteoir";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "bréagach";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Tugann sé fíor nó bréagach ar ais.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "fíor";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://en.wikipedia.org/wiki/Inequality_(mathematics)";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Fill ar ais fíor má tá an dá ionchur cothrom lena chéile.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Tabhair fíor ar ais má tá an chéad ionchur níos mó ná an dara hionchur.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Tabhair fíor ar ais má tá an chéad ionchur níos mó ná nó cothrom leis an dara hionchur.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Tabhair fíor ar ais má tá an chéad ionchur níos lú ná an dara hionchur.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Tabhair fíor ar ais má tá an chéad ionchur níos lú ná nó cothrom leis an dara hionchur.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Fill ar ais fíor mura bhfuil an dá ionchur cothrom lena chéile.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "ní %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Tugann fíor ar ais má tá an t-ionchur bréagach. Tugann bréag ar ais má tá an t-ionchur fíor.";
Blockly.Msg["LOGIC_NULL"] = "nialas";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Tugann sé nialas ar ais.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "agus";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "nó";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Tabhair fíor ar ais má tá an dá ionchur fíor.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Tabhair fíor ar ais má tá ceann amháin ar a laghad de na hionchuir fíor.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "tástáil";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "más bréagach";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "más fíor";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Seiceáil an coinníoll i 'test'. Más fíor an coinníoll, seolann sé an luach 'if true' ar ais; murach sin seolann sé an luach 'if false' ar ais.";
Blockly.Msg["MAC_OS"] = "macOS";
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://en.wikipedia.org/wiki/Arithmetic";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Tabhair suim an dá uimhir ar ais.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Tabhair ar ais cóimheas an dá uimhir.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Tabhair ar ais an difríocht idir an dá uimhir.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Tabhair toradh an dá uimhir ar ais.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Tabhair an chéad uimhir ar ais go cumhacht an dara huimhir.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 de X:%1 Y:%2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Tabhair arctangant an phointe (X, Y) ar ais i gcéimeanna ó -180 go 180.";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "athrú %1 faoi %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Cuir uimhir leis an athróg '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://en.wikipedia.org/wiki/Mathematical_constant";  // untranslated
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Tabhair ceann de na tairiseacha coitianta ar ais: π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), nó ∞ (éigríoch).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "srian %1 íseal %2 ard %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Srian a chur ar uimhir chun go mbeidh sí idir na teorainneacha sonraithe (san áireamh).";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "atá inroinnte ar";
Blockly.Msg["MATH_IS_EVEN"] = "is cothrom";
Blockly.Msg["MATH_IS_NEGATIVE"] = "is diúltach";
Blockly.Msg["MATH_IS_ODD"] = "is corr";
Blockly.Msg["MATH_IS_POSITIVE"] = "is dearfach";
Blockly.Msg["MATH_IS_PRIME"] = "is príomh";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Seiceáil an uimhir chothrom, corr, príomhuimhir, iomlán, dearfach, diúltach í, nó an féidir í a roinnt ar uimhir áirithe. Tugann sé fíor nó bréagach ar ais.";
Blockly.Msg["MATH_IS_WHOLE"] = "is iomlán";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://en.wikipedia.org/wiki/Modulo_operation";  // untranslated
Blockly.Msg["MATH_MODULO_TITLE"] = "fuílleach %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Faigh an fuílleach ar ais ó roinnt an dá uimhir.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://en.wikipedia.org/wiki/Number";  // untranslated
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Uimhir.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "meán an liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "uasmhéid an liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "meán an liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "íosmhéid den liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "modhanna liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "mír randamach den liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "diall caighdeánach liosta";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "suim an liosta";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Tabhair ar ais meán (meán uimhriúil) na luachanna uimhriúla sa liosta.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Tabhair an uimhir is mó sa liosta ar ais.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Tabhair an meánuimhir sa liosta ar ais.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Tabhair an uimhir is lú sa liosta ar ais.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Tabhair liosta ar ais de na míreanna is coitianta sa liosta.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Tabhair eilimint randamach ar ais ón liosta.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Tabhair ar ais an diall caighdeánach den liosta.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Tabhair ar ais suim na n-uimhreacha go léir sa liosta.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "codán randamach";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Tabhair codán randamach ar ais idir 0.0 (san áireamh) agus 1.0 (gan áireamh).";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "slánuimhir randamach ó %1 go %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Tabhair slánuimhir randamach ar ais idir an dá theorainn shonraithe, an líon sin san áireamh.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "babhta";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "babhtaigh síos";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "babhta suas";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Babhtaigh uimhir suas nó síos.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://en.wikipedia.org/wiki/Square_root";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "absalóideach";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "fréamh chearnach";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Tabhair luach absalóideach uimhreach ar ais.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Cuir e ar ais i gcumhacht uimhreach.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Tabhair ar ais logartam nádúrtha uimhir.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Tabhair ar ais logartam bonn 10 uimhir.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Tabhair ar ais diúltú uimhreach.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Cuir 10 ar ais i gcumhacht uimhreach.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Tabhair fréamh chearnach uimhre ar ais.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "cos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://en.wikipedia.org/wiki/Trigonometric_functions";  // untranslated
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Tabhair arccosín uimhir ar ais.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Tabhair arcsín uimhir ar ais.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Tabhair arc-tangánt uimhreach ar ais.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Tabhair ar ais cosíneas céime (ní raidian).";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Tabhair ar ais sine céime (ní raidian).";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Tabhair tadhlaí céime ar ais (ní raidian).";
Blockly.Msg["MOVE_BLOCK"] = "Bog Bloc";
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Crúthaigh dath athróige";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Crúthaigh athróg uimhire";
Blockly.Msg["NEW_STRING_VARIABLE"] = "Crúthaigh sraing athróige";
Blockly.Msg["NEW_VARIABLE"] = "Crúthaigh athróg";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Ainm athróg nua:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Cineál Athróige Nua:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "⌥ Option";
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Greamaigh";
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "ráitis cheadaithe";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "le:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Rith an fheidhm sainmhínithe ag an úsáideoir '%1'.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Rith an fheidhm shainithe ag an úsáideoir '%1' agus bain úsáid as a haschur.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "le:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Ní féidir an fheidhm sainmhínithe ag an úsáideoir '%1' a rith mar go bhfuil an bloc sainmhínithe díchumasaithe.";
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Cruthaigh '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Déan cur síos ar an fheidhm seo...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "déan rud éigin";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "go";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Cruthaíonn sé feidhm gan aon aschur.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "filleadh";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Cruthaíonn feidhm le haschur.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Rabhadh: Tá paraiméadair dhúblacha sa fheidhm seo.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Sainmhíniú feidhme aibhsithe";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Más fíor luach, cuirtear an dara luach ar ais.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Rabhadh: Ní féidir an bloc seo a úsáid ach amháin laistigh de shainmhíniú feidhme.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "ainm ionchuir:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Cuir ionchur leis an bhfeidhm.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "ionchuir";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Cuir ionchuir leis an bhfeidhm seo, bain iad, nó athordaigh iad.";
Blockly.Msg["REDO"] = "Athdhéan";
Blockly.Msg["REMOVE_COMMENT"] = "Bain amach an ráiteas";
Blockly.Msg["RENAME_VARIABLE"] = "Ath-ainmnigh athróg";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Ath-ainmnigh %1 uilig go:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Nascleanúint cóid";
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Eagarthóireacht";
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "Ginearálta";
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "le %1 cuir téacs %2 leis";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Cuir roinnt téacs leis an athróg '%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "i gcás beag";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "chuig Cás Teidil";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "go CÁS UACHTARACH";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Seol cóip den téacs ar ais i gcás difriúil.";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "faigh an chéad litir";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "faigh litir # ón deireadh";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "faigh litir #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "faigh an litir dheireanach";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "faigh litir randamach";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "i dtéacs %1 %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Tugann sé an litir ar ais ag an suíomh sonraithe.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "comhaireamh %1 i %2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Comhair cé mhéad uair a tharlaíonn téacs áirithe laistigh de théacs eile.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Cuir mír leis an téacs.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "téigh i";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Cuir rannóga leis, bain iad, nó athordaigh iad chun an bloc téacs seo a athchumrú.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "go dtí an litir # ón deireadh";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "go litir #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "go dtí an litir dheireanach";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "sa téacs";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "faigh fo-theaghrán ón gcéad litir";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "faigh fo-theaghrán ó litir # ón deireadh";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "faigh fo-theaghrán ó litir #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Tugann sé cuid shonraithe den téacs ar ais.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "aimsigh an chéad uair a tharlaíonn téacs";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "aimsigh an uair dheireanach den téacs";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "sa téacs %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Seolann sé innéacs an chéad/an uair dheireanaigh den chéad téacs sa dara téacs ar ais. Seolann sé %1 ar ais mura bhfuil téacs le fáil.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "Tá %1 folamh";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Tugann sé fíor ar ais má tá an téacs a cuireadh ar fáil folamh.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "téacs a chruthú le";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Cruthaigh píosa téacs trí aon líon míreanna a cheangal le chéile.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "fad %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Tugann sé seo líon na litreacha (lena n-áirítear spásanna) sa téacs a cuireadh ar fáil ar ais.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "priontáil %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Priontáil an téacs, an uimhir nó an luach eile sonraithe.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Iarr uimhir ar an úsáideoir.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Iarr téacs ar an úsáideoir.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "iarr uimhir le teachtaireacht";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "iarr téacs le teachtaireacht";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "cuir %2 in ionad %1 i %3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Cuir téacs éigin in ionad gach áit a bhfuil sé le feiceáil laistigh de théacs eile.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "droim ar ais %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Athraíonn sé ord na gcarachtar sa téacs.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://en.wikipedia.org/wiki/String_(computer_science)";  // untranslated
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Litir, focal, nó líne téacs.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "bearr spásanna ó dhá thaobh an";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "bearr spásanna ón taobh clé de";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "bearr spásanna ón taobh deas de";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Tabhair cóip den téacs ar ais agus na spásanna bainte as ceann amháin nó an dá cheann.";
Blockly.Msg["TODAY"] = "inniú";
Blockly.Msg["UNDO"] = "Cealaigh";
Blockly.Msg["UNKNOWN"] = "Anaithnid";
Blockly.Msg["UNNAMED_KEY"] = "gan ainm";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "mír";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Cruthaigh 'tacar %1'";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Tugann sé luach an athróg seo ar ais.";
Blockly.Msg["VARIABLES_SET"] = "socraigh %1 go %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Cruthaigh 'faigh %1'";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Socraíonn sé an athróg seo le bheith cothrom leis an ionchur.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Tá athróg ainmnithe %1 ann cheana féin.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Tá athróg darbh ainm '%1' ann cheana féin do chíneáil eile: '%2'";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "Tá athróg darb ainm '%1' ann cheana féin mar pharaiméadar sa nós imeachta '%2'.";
Blockly.Msg["WINDOWS"] = "Windows";
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Abair rud éigin...";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
