// blockly/msg/ko.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "주석 추가";
Blockly.Msg["ALT_KEY"] = "Alt";
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "%2 뒤로 %1 이동 중.";
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "%2 주변으로 %1 이동 중.";
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "%2 앞으로 %1 이동 중.";
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "이동이 취소되었습니다.";
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "%2 안으로 %1 이동 중.";
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%2의 %1";
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "%2한테로 %1 이동 중.";
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "작업 공간에서 %1 이동 중.";
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "else if 추가";
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "입력 추가";
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "목록 항목 추가";
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "텍스트 추가";
Blockly.Msg["ARIA_LABEL_BUTTON"] = "버튼";
Blockly.Msg["ARIA_LABEL_COMMENT"] = "주석";
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "주석 접기";
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "주석 펼치기";
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1도";
Blockly.Msg["ARIA_LABEL_HEADING"] = "문단 제목";
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "else if 제거";
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "입력 제거";
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "목록 항목 제거";
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "텍스트 제거";
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "각도";
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "픽셀 이미지";
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "체크 상자";
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "색상";
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "날짜";
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "드롭다운";
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "격자 드롭다운";
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "이미지";
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "입력";
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "숫자";
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "텍스트";
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "입력 이름";
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "함수 이름";
Blockly.Msg["BACKSPACE_KEY"] = "백스페이스";
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "시작 %1";
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "시작 더미";
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "접힘";
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "컨테이너";
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "비활성화됨";
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "%1개 분기가 있음";
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "입력값이 있음";
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "입력값들이 있음";
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "대체 가능";
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "블록 %1개 더미";
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "서술문";
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 분류";
Blockly.Msg["BLOCK_LABEL_VALUE"] = "값";
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "주석: %1";
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "버블";
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "경고: %1";
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "'%2' 함수 정의의 일부이기 때문에 '%1' 변수를 삭제할 수 없습니다";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";
Blockly.Msg["CHANGE_VALUE_TITLE"] = "값 바꾸기:";
Blockly.Msg["CHROME_OS"] = "크롬OS";
Blockly.Msg["CLEAN_UP"] = "블록 정리";
Blockly.Msg["CLOSE"] = "닫기";
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "접힌 블록이 경고를 포함하고 있습니다.";
Blockly.Msg["COLLAPSE_ALL"] = "블록 축소";
Blockly.Msg["COLLAPSE_BLOCK"] = "블록 축소";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "색 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "색 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "비율";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "혼합";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "두 색을 주어진 비율로 혼합 (0.0 - 1.0)";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://ko.wikipedia.org/wiki/색";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "팔레트에서 색을 고릅니다";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "무작위 색상";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "무작위로 색을 고릅니다.";
Blockly.Msg["COLOUR_RGB_BLUE"] = "파랑";
Blockly.Msg["COLOUR_RGB_GREEN"] = "초록";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "빨강";
Blockly.Msg["COLOUR_RGB_TITLE"] = "색";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "빨강,파랑,초록의 값을 이용하여 색을 만드십시오. 모든 값은 0과 100 사이에 있어야 합니다.";
Blockly.Msg["COMMAND_KEY"] = "명령";
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ 메뉴";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "반복 중단";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "다음 반복";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "현재 반복 실행 블럭을 빠져나갑니다.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "나머지 반복 부분을 더 이상 실행하지 않고, 다음 반복을 수행합니다.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "경고 : 이 블록은 반복 실행 블럭 안에서만 사용됩니다.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "각 항목에 대해 %1 목록으로 %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "목록 안에 들어있는 각 항목들을, 순서대로 변수 '%1' 에 한 번씩 저장시키고, 그 때 마다 명령을 실행합니다.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";
Blockly.Msg["CONTROLS_FOR_TITLE"] = "%1을 사용해 %2 부터 %3 까지 %4 씩";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "변수 '%1'이(가) 지정된 간격으로 시작 번호에서 끝 번호까지 세는 동시에 지정된 블록을 실행하게 하세요.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "\"만약\" 블럭에 조건 검사를 추가합니다.";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "\"만약\" 블럭의 마지막에, 모든 검사 결과가 거짓인 경우 실행할 부분을 추가합니다.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "섹션을 추가, 제거하거나 순서를 변경하여 이 if 블럭을 재구성합니다.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "아니라면";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "다른 경우";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "만약";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "조건식의 계산 결과가 참이면, 명령을 실행합니다.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "조건식의 계산 결과가 참이면, 첫 번째 블럭의 명령을 실행하고, 그렇지 않으면 두 번째 블럭의 명령을 실행합니다.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "첫 번째 조건식의 계산 결과가 참이면, 첫 번째 블럭의 명령을 실행하고, 두 번째 조건식의 계산 결과가 참이면, 두 번째 블럭의 명령을 실행합니다.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "첫 번째 조건식의 계산 결과가 참이면, 첫 번째 블럭의 명령을 실행하고, 두 번째 조건식의 계산 결과가 참이면, 두 번째 블럭의 명령을 실행하고, ... , 어떤 조건식의 계산 결과도 참이 아니면, 마지막 블럭의 명령을 실행합니다.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://ko.wikipedia.org/wiki/For_루프";
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "하기";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "%1회 반복";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "여러 번 반복해 명령들을 실행합니다.";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "다음까지 반복";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "동안 반복";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "값이 거짓인 동안 명령문을 실행합니다.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "값이 참일 때, 명령들을 실행합니다.";
Blockly.Msg["CONTROL_KEY"] = "컨트롤";
Blockly.Msg["COPY_SHORTCUT"] = "복사";
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "현재 블록: %1";
Blockly.Msg["CUT_SHORTCUT"] = "잘라내기";
Blockly.Msg["DELETE_ALL_BLOCKS"] = "모든 블록 %1개를 삭제하겠습니까?";
Blockly.Msg["DELETE_BLOCK"] = "블록 삭제";
Blockly.Msg["DELETE_KEY"] = "Delete";
Blockly.Msg["DELETE_VARIABLE"] = "'%1' 변수를 삭제합니다";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "'%2' 변수에서 %1을(를) 삭제하시겠습니까?";
Blockly.Msg["DELETE_X_BLOCKS"] = "블록 %1개 삭제";
Blockly.Msg["DIALOG_CANCEL"] = "취소";
Blockly.Msg["DIALOG_OK"] = "확인";
Blockly.Msg["DISABLE_BLOCK"] = "블록 비활성화";
Blockly.Msg["DUPLICATE_BLOCK"] = "중복";
Blockly.Msg["DUPLICATE_COMMENT"] = "주석 복제";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "차단 내용 편집";
Blockly.Msg["ENABLE_BLOCK"] = "블록 활성화";
Blockly.Msg["END_KEY"] = "End";
Blockly.Msg["ENTER_KEY"] = "Enter";
Blockly.Msg["ESCAPE"] = "ESC";
Blockly.Msg["EXPAND_ALL"] = "블록 확장";
Blockly.Msg["EXPAND_BLOCK"] = "블록 확장";
Blockly.Msg["EXTERNAL_INPUTS"] = "외부 입력";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "체크됨";
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "체크되지 않음";
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "%1 편집";
Blockly.Msg["FIELD_LABEL_EMPTY"] = "비었음";
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "옵션 %1";
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "변수 '%1'";
Blockly.Msg["HELP"] = "도움말";
Blockly.Msg["HELP_PROMPT"] = "키보드 조작법에 대한 도움말을 보려면 %1을 누르세요.";
Blockly.Msg["HOME_KEY"] = "Home";
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "주석 열기";
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "주석 닫기";
Blockly.Msg["ICON_LABEL_DEFAULT"] = "아이콘";
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "이 블록을 편집";
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "블록 편집기 닫기";
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "경고 열기";
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "경고 닫기";
Blockly.Msg["INLINE_INPUTS"] = "내부 입력";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "조건";
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "첫 번째 조건";
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "두 번째 조건";
Blockly.Msg["INPUT_LABEL_EMPTY"] = "비었음";
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "끝 %1";
Blockly.Msg["INPUT_LABEL_INDEX"] = "입력 %1";
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "값 %1";
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "구분자";
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "종료 위치";
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "나눌 텍스트";
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "목록 안에서의 위치";
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "반복할 값";
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "반복할 횟수";
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "시작 위치";
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "이어붙일 텍스트";
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "바꿀 목록";
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "확인할 목록";
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "설정할 값";
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "증가분";
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "시작 숫자";
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "반복할 횟수";
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "종료 숫자";
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "제한할 숫자";
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "피제수";
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "제수";
Blockly.Msg["INPUT_LABEL_NUMBER"] = "숫자";
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "첫 번째 숫자";
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x 좌표";
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y 좌표";
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "두 번째 숫자";
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "숫자 목록";
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "최대값";
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "최소값";
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "확인할 숫자";
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "서술문 위치";
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "추가할 값";
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "종료 위치";
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "값 %1";
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "문자 위치";
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "메시지";
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "시작 위치";
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "바꿀 텍스트";
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "확인할 텍스트";
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "찾을 텍스트";
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "바꿀 텍스트";
Blockly.Msg["INPUT_LABEL_VALUE"] = "값 위치";
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "첫 번째 값";
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "두 번째 값";
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "설정할 값";
Blockly.Msg["INSERT_KEY"] = "Insert";
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "%1을(를) 사용하여 블록 안을 탐색하세요.";
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "화살표 키를 사용하여 이동한 다음, %1 키를 눌러 위치를 확정하세요.";
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "복사되었습니다. %1를 눌러 붙여넣으세요.";
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "잘라내었습니다. %1를 눌러 붙여넣으세요.";
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "화살표 키를 사용하여 블록을 탐색하거나, %1을(를) 눌러 다음 문단 제목으로 이동하세요.";
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "%1 키를 누른 상태에서 화살표 키를 사용하여 자유롭게 이동한 다음, %2 키를 눌러 위치를 확정하세요.";
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "화살표 키를 사용하여 탐색하세요.";
Blockly.Msg["LINUX"] = "리눅스";
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "빈 목록 만들기";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "데이터 레코드가 없는, 길이가 0인 목록을 반환합니다.";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "목록";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "섹션을 추가, 제거하거나 순서를 변경하여 이 목록 블럭을 재구성합니다.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "목록 만들기";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "항목을 목록에 추가합니다.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "원하는 수의 항목들로 목록을 만듭니다.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "첫 번째";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "끝에서부터 # 번째";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "가져오기";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "잘라 내기";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "마지막";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "임의로";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "삭제";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "목록의 첫 번째 항목을 반환합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "목록에서 특정 위치의 항목을 반환합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "목록의 마지막 항목을 찾아 반환합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "목록의 랜덤한 항목을 반환합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "목록의 첫 번째 항목을 반환한 후, 그 항목을 목록에서 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "목록의 특정 위치에 있는 항목을 제거하고 반환합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "목록의 마지막 항목을 반환한 후, 그 항목을 목록에서 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "목록의 임의의 항목을 반환한 후, 그 항목을 목록에서 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "목록에서 첫 번째 항목을 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "목록에서 특정 위치의 항목을 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "목록에서 마지막 항목을 찾아 삭제합니다.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "목록에서 랜덤하게 항목을 삭제합니다.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "끝에서부터 # 번째로";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "앞에서부터 # 번째로";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "마지막으로";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "앞에서부터 하위 목록 추출";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "뒤에서부터 #번째 위치부터 하위 목록 추출";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "#번째 위치부터 하위 목록 추출";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "목록의 특정 부분에 대한 복사본을 만듭니다.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1은(는) 마지막 항목입니다.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1은 첫 번째 항목입니다.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "처음으로 나타난 위치";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "마지막으로 나타난 위치";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "목록에서 항목이 처음 또는 마지막으로 발생한 색인 위치를 반환합니다. 항목이 없으면 %1을 반환합니다.";
Blockly.Msg["LISTS_INLIST"] = "목록";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1이 비어 있음";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "목록이 비었을 때 참을 반환합니다.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "%1의 길이";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "목록의 길이를 반환합니다.";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "%1을 %2번 반복한 목록을 만들기";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "지정된 값을, 지정된 개수 만큼 넣어, 목록을 만듭니다.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "%1 뒤집기";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "목록의 복사본을 뒤집습니다.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "에";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "에서 원하는 위치에 삽입";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "에서 설정";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "항목을 목록의 처음 위치에 삽입합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "목록의 특정 위치에 항목을 삽입합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "목록의 마지막에 항목을 추가합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "목록의 임의의 위치에 항목을 삽입합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "목록의 첫 번째 항목으로 설정합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "목록의 특정 위치에 있는 항목으로 설정합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "목록의 마지막 항목으로 설정합니다.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "목록에서 임의의 위치의 항목으로 설정합니다.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "오름차순";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "내림차순";
Blockly.Msg["LISTS_SORT_TITLE"] = "정렬 %1 %2 %3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "목록의 사본을 정렬합니다.";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "알파벳순 (대소문자 구분 안 함)";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "숫자순";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "알파벳순";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "텍스트에서 목록 만들기";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "목록에서 텍스트 만들기";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "구분 기호로 구분하여 텍스트 목록을 하나의 텍스트에 병합합니다.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "각 속보, 텍스트의 목록들에서 텍스트를 분할합니다.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "분리와";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "거짓";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "참 혹은 거짓 모두 반환합니다.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "참";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "같음";
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "크거나 같음";
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "큼";
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://ko.wikipedia.org/wiki/부등식";
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "작거나 같음";
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "작음";
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "같지 않음";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "두 값이 같을 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "첫 번째 값이 두 번째 값보다 클 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "첫 번째 값이 두 번째 값보다 크거나 같을 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "첫 번째 값이 두 번째 값보다 작을 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "첫 번째 값이 두 번째 값보다 작거나 같을 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "두 값이 서로 다를 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "%1가 아님";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "입력값이 거짓이라면 참을 반환합니다. 참이라면 거짓을 반환합니다.";
Blockly.Msg["LOGIC_NULL"] = "빈 값";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://ko.wikipedia.org/wiki/널러블_타입";
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "빈 값을 반환합니다.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "그리고";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";
Blockly.Msg["LOGIC_OPERATION_OR"] = "또는";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "두 값이 모두 참 값일 경우, 참 값을 반환합니다.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "적어도 하나의 값이 참일 경우 참을 반환합니다.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "테스트";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://ko.wikipedia.org/wiki/%3F:";
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "만약 거짓이라면";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "만약 참이라면";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "'test'의 조건을 검사합니다. 조건이 참이면 'if true' 값을 반환합니다. 거짓이면 'if false' 값을 반환합니다.";
Blockly.Msg["MAC_OS"] = "macOS";
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "더하기";
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://ko.wikipedia.org/wiki/산술";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "두 수의 합을 반환합니다.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "두 수의 나눈 결과를 반환합니다.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "두 수간의 차이를 반환합니다.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "두 수의 곱을 반환합니다.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "첫 번째 수를 두 번째 수 만큼, 거듭제곱 한 결과값을 반환합니다.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "X:%1 Y:%2의 atan2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "점 (X, Y)의 아크탄젠트를 -180에서 180까지 도 단위로 반환합니다.";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "%1 을 %2 만큼 바꾸기";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "변수 '%1' 에 저장되어있는 값에, 어떤 수를 더해, 변수에 다시 저장합니다.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "황금비";
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://ko.wikipedia.org/wiki/수학_상수";
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "무한대";
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "파이";
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "1/2의 제곱근";
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "2의 제곱근";
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "일반적인 상수 값들 중 하나를 반환합니다. : π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), 또는 ∞ (무한대).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://ko.wikipedia.org/wiki/클램핑_(그래픽)";
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "%1의 값을, 최소 %2 최대 %3으로 조정";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "어떤 수를, 특정 범위의 값이 되도록 강제로 조정합니다.";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "나누기";
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "가 다음 수로 나누어 떨어지면 :";
Blockly.Msg["MATH_IS_EVEN"] = "가 짝수이면";
Blockly.Msg["MATH_IS_NEGATIVE"] = "가 음(-)수 이면";
Blockly.Msg["MATH_IS_ODD"] = "가 홀수이면";
Blockly.Msg["MATH_IS_POSITIVE"] = "가 양(+)수 이면";
Blockly.Msg["MATH_IS_PRIME"] = "가 소수이면";
Blockly.Msg["MATH_IS_TOOLTIP"] = "어떤 수가 짝수, 홀수, 소수, 정수, 양수, 음수, 나누어 떨어지는 수 인지 검사해 그 결과를 반환합니다. 참 또는 거짓 값을 반환합니다.";
Blockly.Msg["MATH_IS_WHOLE"] = "가 정수이면";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://ko.wikipedia.org/wiki/모듈로";
Blockly.Msg["MATH_MODULO_TITLE"] = "%1 ÷ %2의 나머지";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "첫 번째 수를 두 번째 수로 나눈 나머지 값을 반환합니다.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "x";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "곱하기";
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://ko.wikipedia.org/wiki/수_(수학)";
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "수";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "평균값";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "최대값";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "최대";
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "중간값";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "최소값";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "최소";
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "가장 여러 개 있는 값";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "목록의 임의 항목";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "표준 편차";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "합";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "목록에 들어있는 숫자들의 평균(산술 평균)을 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "목록에 들어있는 숫자들 중 가장 큰 숫자를 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "목록에 들어있는 숫자들의 중간값을 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "리스트에 들어있는 숫자들 중 가장 작은 숫자를 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "목록에 들어있는 항목들 중에서, 가장 여러 번 들어있는 항목(들)을 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "목록에서 임의의 항목을 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "목록의 표준 편차를 반환합니다.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "목록에 들어있는 값들의 총합을 반환합니다.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "제곱수";
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://ko.wikipedia.org/wiki/난수 발생기";
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "임의의 분수";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "0.0 (포함함) 과 1.0 (포함하지 않음) 사이의 임의의 분수 값을 반환합니다.";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://ko.wikipedia.org/wiki/난수 발생기";
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "랜덤정수(%1<= n <=%2)";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "지정된 두 범위(양쪽 포함) 내의 임의의 정수를 반환합니다.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://ko.wikipedia.org/wiki/반올림";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "반올림";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "버림";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "올림";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "숫자를 올림 혹은 버림합니다.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://ko.wikipedia.org/wiki/제곱근";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "절대값";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "절댓값";
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e제곱수";
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "자연로그";
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "밑이 10인 로그";
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "음수";
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10제곱수";
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "제곱근";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "숫자의 절대값을 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "e의 거듭제곱 값을 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "숫자의 자연로그 값을 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "숫자의 밑이 10인 로그 값을 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "숫자의 부호를 반대로 하여 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "10의 거듭제곱 값을 반환합니다.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "숫자의 제곱근을 반환합니다.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "빼기";
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "역코사인";
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "역사인";
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "역탄젠트";
Blockly.Msg["MATH_TRIG_COS"] = "cos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "코사인";
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://ko.wikipedia.org/wiki/삼각함수";
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "사인";
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "탄젠트";
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "어떤 수에 대한 arccosine 값을 반환합니다.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "어떤 수에 대한 arcsine 값을 반환합니다.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "어떤 수에 대한 arctangent 값을 반환합니다.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "각도의 코사인을 반환합니다. (라디안 아님)";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "각도의 사인을 반환합니다. (라디안 아님)";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "각도의 탄젠트를 반환합니다. (라디안 아님)";
Blockly.Msg["MOVE_BLOCK"] = "이동 블록";
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "색 변수 만들기...";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "숫자 변수 만들기....";
Blockly.Msg["NEW_STRING_VARIABLE"] = "문자열 변수 만들기...";
Blockly.Msg["NEW_VARIABLE"] = "변수 만들기...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "새 변수 이름:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "새 변수 유형:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "현재 블록에는 부모 블록이 없습니다";
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "옵션";
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "부모 블록: %1";
Blockly.Msg["PASTE_SHORTCUT"] = "붙여넣기";
Blockly.Msg["PAUSE_KEY"] = "Pause";
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "서술 허가";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "사용:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://ko.wikipedia.org/wiki/함수_(프로그래밍)";
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "미리 정의해 둔 '%1' 함수를 실행합니다.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://ko.wikipedia.org/wiki/함수_(프로그래밍)";
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "미리 정의해 둔 '%1' 함수를 실행하고, 함수를 실행한 결과 값을 사용합니다.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "사용:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "정의 블록이 비활성화되었기 때문에 사용자 지정 함수 '%1'를 실행할 수 없습니다.";
Blockly.Msg["PROCEDURES_CREATE_DO"] = "'%1' 만들기";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "이 함수를 설명하세요...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://ko.wikipedia.org/wiki/함수_(컴퓨터_과학)";
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "함수 이름";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "함수";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "출력값이 없는 함수를 생성합니다.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://ko.wikipedia.org/wiki/함수_(컴퓨터_과학)";
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "다음을 반환함";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "출력값을 가지는 함수를 생성합니다.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "경고: 이 함수에는, 같은 이름을 사용하는 매개변수들이 있습니다.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "함수 정의 찾기";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "값이 참이라면, 두 번째 값을 반환합니다.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "경고: 이 블럭은, 함수 정의 블럭 안에서만 사용할 수 있습니다.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "매개변수:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "함수에 값을 더합니다.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "매개변수들";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "이 함수를 추가, 삭제, 혹은 재정렬합니다.";
Blockly.Msg["REDO"] = "다시 실행";
Blockly.Msg["REMOVE_COMMENT"] = "주석 제거";
Blockly.Msg["RENAME_VARIABLE"] = "'%1' 변수 이름 바꾸기";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "'%1' 변수 이름을 바꾸기:";
Blockly.Msg["RESET_ZOOM"] = "확대/축소 초기화";
Blockly.Msg["SCREENREADER_HINT"] = "화살표 키를 사용하여 탐색하세요. %1을(를) 눌러 읽기 접근성 모드를 켜거나 끌 수 있습니다.";
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "읽기 모드가 꺼져 있습니다, 켜시려면 %1을(를) 누르세요";
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "읽기 모드가 켜져 있습니다, 끄시려면 %1을(를) 누르세요";
Blockly.Msg["SHIFT_KEY"] = "시프트";
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "이동 중단";
Blockly.Msg["SHORTCUTS_CLEANUP"] = "작업 공간 치우기";
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "코드 둘러보기";
Blockly.Msg["SHORTCUTS_DELETE"] = "삭제";
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "블록 연결 해제";
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "복제";
Blockly.Msg["SHORTCUTS_EDITING"] = "편집";
Blockly.Msg["SHORTCUTS_ESCAPE"] = "나가기";
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "자세한 정보 표시하기";
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "이동 완료";
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "도구 상자에 초점 맞추기";
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "작업 공간에 초점 맞추기";
Blockly.Msg["SHORTCUTS_GENERAL"] = "일반";
Blockly.Msg["SHORTCUTS_INFORMATION"] = "정보 표시하기";
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "아래쪽으로 이동";
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "왼쪽으로 이동";
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "오른쪽으로 이동";
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "위쪽으로 이동";
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "다음 문단 제목";
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "다음 더미";
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "편집 혹은 확인";
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "이전 문단 제목";
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "이전 더미";
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "메뉴 표시";
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "툴팁 표시";
Blockly.Msg["SHORTCUTS_START_MOVE"] = "이동 시작";
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "더미 이동 시작";
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "읽기 모드 전환";
Blockly.Msg["SPACE_KEY"] = "스페이스키";
Blockly.Msg["TAB_KEY"] = "탭";
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "%1에 다음 내용 추가하기 %2";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "'%1' 변수의 끝에 일부 텍스트를 추가합니다.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "소문자로";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "첫 문자만 대문자로";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "대문자로";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "텍스트의 복사본을 대소문자를 변경해 반환합니다.";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "에서, 첫 번째 문자 얻기";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "끝에서부터 # 번째 위치의 문자 얻기";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "에서, 앞에서부터 # 번째 위치의 문자 얻기";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "에서, 마지막 문자 얻기";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "에서, 랜덤하게 한 문자 얻기";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "텍스트 %1 %2에서";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "특정 위치의 문자를 반환합니다.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "%2에서 %1 숫자 세기";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "다른 어떤 텍스트에서 어떤 텍스트가 나타난 횟수를 셉니다.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "텍스트에 항목을 추가합니다.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "가입";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "섹션을 추가, 제거하거나 순서를 변경하여 이 텍스트 블럭을 재구성합니다.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "끝에서부터의 문자 번호";
Blockly.Msg["TEXT_FROM_START_ARIA"] = "문자 번호";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "끝에서부터 # 번째 문자까지";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "# 번째 문자까지";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "마지막 문자까지";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "텍스트";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "에서, 처음부터 얻어냄";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "끝에서부터 # 번째부터 얻어냄";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "처음부터 # 번째 문자부터 얻어냄";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "텍스트의 특정 부분을 반환합니다.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "에서 다음 텍스트가 처음으로 나타난 위치 찾기:";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "에서 다음 텍스트가 마지막으로 나타난 위치 찾기:";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "텍스트 %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "두 번째 텍스트에서 첫 번째 텍스트가 처음 또는 마지막으로 발생한 색인 위치를 반환합니다. 텍스트가 없으면 %1을 반환합니다.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1이 비어 있음";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "입력된 텍스트가 비어있으면 참 값을 반환합니다.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "텍스트 만들기";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "여러 개의 항목들을 연결해, 새로운 문장을 만듭니다.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "다음의 문자 개수 %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "입력된 텍스트의 문자 개수를 반환합니다 (공백문자 포함).";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "다음 내용 출력 %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "원하는 텍스트, 수, 값 등을 출력합니다.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "수에 대해 사용자의 입력을 받습니다.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "텍스트에 대해 사용자의 입력을 받습니다.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "메시지를 활용해 수 입력";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "메시지를 활용해 텍스트 입력";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "%3에서 %2을(를) %1(으)로 바꾸기";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "다른 텍스트 내에서 일부 텍스트의 모든 발생을 치환합니다.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "%1 뒤집기";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "텍스트 안의 문자의 순서를 반전시킵니다.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://ko.wikipedia.org/wiki/문자열";
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "문자, 단어 또는 문장입니다.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "양쪽의 공백 문자 제거";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "왼쪽의 공백 문자 제거";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "오른쪽의 공백 문자 제거";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "텍스트의 한쪽 혹은 양쪽에서 공백 문자를 제거해 반환합니다.";
Blockly.Msg["TODAY"] = "오늘";
Blockly.Msg["UNDO"] = "되돌리기";
Blockly.Msg["UNKNOWN"] = "알 수 없음";
Blockly.Msg["UNNAMED_KEY"] = "이름이 없는";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "항목";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "'집합 %1' 만들기";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "변수의 값을 반환합니다.";
Blockly.Msg["VARIABLES_SET"] = "%1를 %2로 설정";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "'%1 값 읽기' 블럭 만들기";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "변수의 값을 입력한 값으로 변경해 줍니다.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "'%1' 변수는 이미 존재합니다.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "'%1' 변수는 다른 유형에 대해 이미 존재합니다: '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "'%1' 매개변수가 이미 '%2' 프로시저의 변수로 존재합니다.";
Blockly.Msg["WINDOWS"] = "윈도";
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "말해 보세요...";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "작업 공간에 블록 더미 %1개%2가 있습니다.";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "작업 공간에 블록 더미 1개%2가 있습니다.";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "블록 더미 0개%2가 있습니다.";
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = "와 주석 %1개";
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = "와 주석 1개";
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "블록 작업 공간입니다. 블록 더미 1개";
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 블록";
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "블록 작업 공간입니다. 블록 더미 %1개";
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "블록 편집기 작업 공간";
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "블록 작업 공간입니다.";
Blockly.Msg["ZOOM_IN"] = "확대";
Blockly.Msg["ZOOM_OUT"] = "축소";
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
