// blockly/msg/ms.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Berikan Komen";
Blockly.Msg["ALT_KEY"] = "Alt";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Can't delete the variable '%1' because it's part of the definition of the function '%2'";  // untranslated
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Ubah nilai:";
Blockly.Msg["CHROME_OS"] = "ChromeOS";  // untranslated
Blockly.Msg["CLEAN_UP"] = "Kemaskan Blok";
Blockly.Msg["CLOSE"] = "Tutup";
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Collapsed blocks contain warnings.";  // untranslated
Blockly.Msg["COLLAPSE_ALL"] = "Lipat Blok²";
Blockly.Msg["COLLAPSE_BLOCK"] = "Lipat Blok";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "warna 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "warna 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "nisbah";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "adun";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Campurkan dua warna sekali pada nisbah yang ditentukan (0.0 - 1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://ms.wikipedia.org/wiki/Warna";
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Pilih satu warna daripada palet.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "warna rawak";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Pilih satu warna secara rawak.";
Blockly.Msg["COLOUR_RGB_BLUE"] = "biru";
Blockly.Msg["COLOUR_RGB_GREEN"] = "hijau";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "merah";
Blockly.Msg["COLOUR_RGB_TITLE"] = "warnakan";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Peroleh satu warna dengan menentukan amaun campuran merah, hijau dan biru. Kesemua nilai haruslah antara 0 hingga 100.";
Blockly.Msg["COMMAND_KEY"] = "Command";  // untranslated
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "hentikan gelung";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "teruskan dengan lelaran gelung seterusnya";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Keluar dari gelung pengandung.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Langkau seluruh gelung yang tinggal dan bersambung dengan lelaran seterusnya.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Amaran: Blok ini hanya boleh digunakan dalam satu gelung.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "untuk setiap perkara %1 dalam senarai %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Untuk setiap perkara dalam senarai, tetapkan pembolehubah '%1' pada perkara, kemudian lakukan beberapa perintah.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "kira dengan %1 dari %2 hingga %3 selang %4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Gunakan pembolehubah '%1' pada nilai-nilai dari nombor pangkal hingga nombor hujung, mengira mengikut selang yang ditentukan, dan lakukan blok-blok yang tertentu.";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Tambah satu syarat kepada blok jika.";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Tambah yang terakhir, alihkan semua keadaan ke blok jika.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Tambah, alih keluar, atau susun semula bahagian-bahagian untuk menyusun semula blok jika.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "lain";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "lain jika";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "jika";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Jika nilai yang benar, lakukan beberapa penyata.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Jika suatu nilai benar, lakukan penyata blok pertama.  Jika tidak, bina penyata blok kedua.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Jika nilai yang pertama adalah benar, lakukan penyata pertama blok.  Sebaliknya, jika nilai kedua adalah benar, lakukan penyata blok kedua.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Jika nilai yang pertama adalah benar, lakukan penyata blok pertama.  Sebaliknya, jika nilai kedua adalah benar, lakukan penyata blok kedua.  Jika tiada nilai adalah benar, lakukan penyata blok terakhir.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://en.wikipedia.org/wiki/For_loop";  // untranslated
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "lakukan";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "ulang %1 kali";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Lakukan perintah berulang kali.";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "ulangi sehingga";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "ulangi apabila";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Lakukan beberapa perintah apabila nilainya palsu (false).";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Lakukan beberapa perintah apabila nilainya benar (true).";
Blockly.Msg["CONTROL_KEY"] = "Control";  // untranslated
Blockly.Msg["COPY_SHORTCUT"] = "Copy";  // untranslated
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Cut";  // untranslated
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Hapuskan kesemua %1 blok?";
Blockly.Msg["DELETE_BLOCK"] = "Hapuskan Blok";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Delete the '%1' variable";  // untranslated
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Delete %1 uses of the '%2' variable?";  // untranslated
Blockly.Msg["DELETE_X_BLOCKS"] = "Hapuskan %1 Blok";
Blockly.Msg["DIALOG_CANCEL"] = "Batalkan";
Blockly.Msg["DIALOG_OK"] = "OK";
Blockly.Msg["DISABLE_BLOCK"] = "Matikan Blok";
Blockly.Msg["DUPLICATE_BLOCK"] = "Pendua";
Blockly.Msg["DUPLICATE_COMMENT"] = "Gandakan Komen";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Edit Block contents";  // untranslated
Blockly.Msg["ENABLE_BLOCK"] = "Hidupkan Blok";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Buka Blok²";
Blockly.Msg["EXPAND_BLOCK"] = "Buka Blok";
Blockly.Msg["EXTERNAL_INPUTS"] = "Input Luaran";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Bantuan";
Blockly.Msg["HELP_PROMPT"] = "Press %1 for help on keyboard controls.";  // untranslated
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Input Sebaris";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Use the arrow keys to move, then %1 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Copied. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Cut. Press %1 to paste.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Hold %1 and use arrow keys to move freely, then %2 to accept the position.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "Wujudkan senarai kosong";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Kembalikan senarai panjang 0, yang tidak mengandungi rekod data";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "senarai";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Tambah, alih keluar, atau susun semula bahagian-bahagian untuk menyusun semula senarai blok.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "wujudkan senarai dengan";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Tambah item ke dalam senarai.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Wujudkan senarai dengan apa jua nombor item.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "pertama";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# dari akhir";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "dapatkan";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "dapat dan alihkan";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "terakhir";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "rawak";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "alihkan";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Kembalikan item pertama dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Kembalikan item dalam kedudukan yang ditetapkan dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Kembalikan item pertama dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Kembalikan item rawak dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Alihkan dan kembalikan item pertama dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Alihkan dan kembalikan item mengikut spesifikasi posisi dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Alihkan dan kembalikan item terakhir dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Alihkan dan kembalikan item rawak dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Alihkan item pertama dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Alihkan item pada posisi mengikut spesifikasi dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Alihkan item terakhir dalam senarai.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Alihkan item rawak dalam senarai.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "ke # dari akhir";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "ke #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "ke akhir";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "dapatkan sub-senarai daripada pertama";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "dapatkan sub-senarai daripada # daripada terakhir";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "dapatkan sub-senarai daripada #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Wujudkan salinan bahagian yang ditentukan dari senarai.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 ialah item terakhir.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 ialah item pertama.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "cari pertama item kejadian";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "cari kejadian akhir item";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Menyatakan indeks kejadian pertama/terakhir item berkenaan dalam senarai. Menyatakan %1 jika item berkenaan tidak ditemui.";
Blockly.Msg["LISTS_INLIST"] = "dalam senarai";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 adalah kosong";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Kembalikan benar jika senarai kosong.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "panjang %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Kembalikan panjang senarai";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "wujudkan senarai dengan item %1 diulangi %2 kali";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Wujudkan senarai yang terdiri daripada nilai berulang mengikut nombor yang ditentukan.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "reverse %1";  // untranslated
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Reverse a copy of a list.";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "sebagai";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "masukkan pada";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "set";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Selit item pada permulaan senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Masukkan item pada posisi yand ditentukan dalam senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Tambahkan item dalam senarai akhir.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Selit item secara rawak di dalam senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Set item pertama dalam senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Masukkan item pada posisi yang ditentukan dalam senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Set item terakhir dalam senarai.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Set item rawak dalam senarai.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "ascending";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "descending";  // untranslated
Blockly.Msg["LISTS_SORT_TITLE"] = "sort %1 %2 %3";  // untranslated
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Sort a copy of a list.";  // untranslated
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "alphabetic, ignore case";  // untranslated
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "numeric";  // untranslated
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "alphabetic";  // untranslated
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "buat senarai dgn teks";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "buat teks drpd senarai";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Cantumkan senarai teks menjadi satu teks, dipecahkan oleh delimiter.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Pecahkan teks kepada senarai teks, berpecah di setiap delimiter.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "dengan delimiter";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "palsu";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Kembalikan samada benar atau palsu.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "benar";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://id.wikipedia.org/wiki/Pertidaksamaan";
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Kembali benar jika kedua-dua input benar antara satu sama lain.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Kembali benar jika input pertama adalah lebih besar daripada input kedua.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Kembali benar jika input pertama adalah lebih besar daripada atau sama dengan input kedua.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Kembali benar jika input pertama adalah lebih kecil daripada input kedua.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Kembali benar jika input pertama adalah lebih kecil daripada atau sama dengan input kedua.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Taip balik benar jika kedua-dua input tidak sama.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "bukan %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "'Benar' akan dibalas jika inputnya salah. 'Salah' akan dibalas jika inputnya benar.";
Blockly.Msg["LOGIC_NULL"] = "null";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Returns null.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "dan";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "atau";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Return true if both inputs are true.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Return true if at least one of the inputs is true.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "test";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "Jika palsu";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "Jika benar";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Check the condition in 'test'. If the condition is true, returns the 'if true' value; otherwise returns the 'if false' value.";
Blockly.Msg["MAC_OS"] = "macOS";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://ms.wikipedia.org/wiki/Aritmetik";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Kembalikan jumlah kedua-dua bilangan.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Taip balik hasil bahagi dua nombor tersebut.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Taip balik hasil tolak dua nombor tersebut.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Taip balik hasil darab dua nombor tersebut.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Return the first number raised to the power of the second number.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 of X:%1 Y:%2";  // untranslated
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Return the arctangent of point (X, Y) in degrees from -180 to 180.";  // untranslated
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://id.wikipedia.org/wiki/Perjumlahan";
Blockly.Msg["MATH_CHANGE_TITLE"] = "perubahan %1 oleh %2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Tambah nombor kepada pembolehubah '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://ms.wikipedia.org/wiki/Pemalar_matematik";
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Return one of the common constants: π (3.141…), e (2.718…), φ (1.618…), sqrt(2) (1.414…), sqrt(½) (0.707…), or ∞ (infinity).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "constrain %1 low %2 high %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Constrain a number to be between the specified limits (inclusive).";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "Boleh dibahagikan dengan";
Blockly.Msg["MATH_IS_EVEN"] = "Adalah genap";
Blockly.Msg["MATH_IS_NEGATIVE"] = "negatif";
Blockly.Msg["MATH_IS_ODD"] = "aneh";
Blockly.Msg["MATH_IS_POSITIVE"] = "adalah positif";
Blockly.Msg["MATH_IS_PRIME"] = "is prime";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Check if a number is an even, odd, prime, whole, positive, negative, or if it is divisible by certain number.  Returns true or false.";
Blockly.Msg["MATH_IS_WHOLE"] = "is whole";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://id.wikipedia.org/wiki/Operasi_modulus";
Blockly.Msg["MATH_MODULO_TITLE"] = "remainder of %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Taip balik baki yang didapat daripada pembahagian dua nombor tersebut.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://ms.wikipedia.org/wiki/Nombor";
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Suatu nombor.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "purata daripada senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "Max senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "Median senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "min dalam senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "jenis senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "Item rawak daripada senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "sisihan piawai bagi senarai";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "Jumlah senarai";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Kembalikan purata (min aritmetik) nilai-nilai angka di dalam senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Pulangkan jumlah terbesar dalam senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Kembalikan nombor median dalam senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Kembalikan nombor terkecil dalam senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Kembali senarai item yang paling biasa dalam senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Kembalikan elemen rawak daripada senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Kembali dengan sisihan piawai daripada senarai.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Kembalikan jumlah semua nombor dalam senarai.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "pecahan rawak";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Kembali sebahagian kecil rawak antara 0.0 (inklusif) dan 1.0 (eksklusif).";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "integer rawak dari %1ke %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Kembalikan integer rawak diantara dua had yang ditentukan, inklusif.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "pusingan";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "Pusingan ke bawah";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "pusingan ke atas";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Bulat nombor yang naik atau turun.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://ms.wikipedia.org/wiki/Punca_kuasa_dua";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "mutlak";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "Punca kuasa dua";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Kembalikan nilai mutlak suatu nombor.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Kembalikan e kepada kuasa nombor.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Kembali dalam logaritma nombor asli.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Kembali logarithm 10 asas nombor.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Kembalikan nombor yang songsang.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Kembalikan 10 kepada kuasa nombor.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Kembalikan punca kuasa nombor.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "acos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "cos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://ms.wikipedia.org/wiki/Fungsi_trigonometri";
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Kembali arccosine beberapa nombor.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Kembalikan arcsince beberapa nombor.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Kembalikan beberapa nombor arctangent.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Kembalikan darjah kosinus (bukan radian).";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Kembalikan darjah sine (bukan radian).";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Kembalikan darjah tangen (bukan radian).";
Blockly.Msg["MOVE_BLOCK"] = "Move Block";  // untranslated
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Cipta pemboleh ubah warna...";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Cipta pemboleh ubah nombor...";
Blockly.Msg["NEW_STRING_VARIABLE"] = "Cipta pemboleh ubah rentetan...";
Blockly.Msg["NEW_VARIABLE"] = "Pembolehubah baru...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Nama pembolehubah baru:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Jenis pemboleh ubah baharu:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "Option";  // untranslated
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Paste";  // untranslated
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "membenarkan kenyataan";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "dengan:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Run the user-defined function '%1'.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Run the user-defined function '%1' and use its output.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "dengan:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Can't run the user-defined function '%1' because the definition block is disabled.";  // untranslated
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Hasilkan '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Terangkan fungsi ini...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "Buat sesuatu";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "Untuk";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Menghasilkan suatu fungsi tanpa output.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "kembali";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Mencipta satu fungsi dengan pengeluaran.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Amaran: Fungsi ini mempunyai parameter yang berganda.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Serlahkan definisi fungsi";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "If a value is true, then return a second value.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Amaran: Blok ini hanya boleh digunakan dalam fungsi definisi.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "Nama input:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Tambah satu input pada fungsi.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "Input-input";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Tambah, alih keluar atau susun semula input pada fungsi ini.";
Blockly.Msg["REDO"] = "Redo";  // untranslated
Blockly.Msg["REMOVE_COMMENT"] = "Padamkan Komen";
Blockly.Msg["RENAME_VARIABLE"] = "Tukar nama pembolehubah...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Tukar nama semua pembolehubah '%1' kepada:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Code navigation";  // untranslated
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Editing";  // untranslated
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "General";  // untranslated
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "to %1 append text %2";  // untranslated
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Append some text to variable '%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "Kepada huruf kecil";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "to Title Case";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "Kepada HURUF BESAR";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Return a copy of the text in a different case.";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "get first letter";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "get letter # from end";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "get letter #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "Dapatkan abjad terakhir";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "Dapatkan abjad rawak";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "in text %1 %2";  // untranslated
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Returns the letter at the specified position.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "count %1 in %2";  // untranslated
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Count how many times some text occurs within some other text.";  // untranslated
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Add an item to the text.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "Sertai";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Add, remove, or reorder sections to reconfigure this text block.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "to letter # from end";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "to letter #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "untuk huruf terakhir";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "dalam teks";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "get substring from first letter";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "get substring from letter # from end";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "get substring from letter #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Returns a specified portion of the text.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "mencari kejadian pertama teks";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "mencari kejadian terakhir teks";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "dalam teks %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Kembalikan Indeks kejadian pertama/terakhir dari teks pertama ke dalam teks kedua.  Kembalikan %1 Jika teks tidak ditemui.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 adalah kosong";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Kembalikan benar jika teks yang disediakan adalah kosong.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "hasilkan teks dengan";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Hasilkan sebahagian teks dengan menghubungkan apa jua nombor item.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "panjang %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Kembalikan jumlah huruf (termasuk ruang) dalam teks yang disediakan.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "cetak %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Cetak teks yang ditentukan, nombor atau nilai lain.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Peringatan kepada pengguna untuk nombor.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Peringatkan pengguna untuk sebahagian teks.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "Prom untuk nombor dengan mesej";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "Prom untuk teks dengan mesej";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "replace %1 with %2 in %3";  // untranslated
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Replace all occurances of some text within some other text.";  // untranslated
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "reverse %1";  // untranslated
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Reverses the order of the characters in the text.";  // untranslated
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://ms.wikipedia.org/wiki/Rentetan";
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Huruf, perkataan, atau baris teks.";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "mengurangkan kawasan dari kedua-dua belah";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "mengurangkan ruang dari sebelah kiri";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "mengurangkan kawasan dari sisi kanan";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Kembali salinan teks dengan ruang yang dikeluarkan daripada satu atau hujung kedua belah.";
Blockly.Msg["TODAY"] = "Hari ini";
Blockly.Msg["UNDO"] = "Undo";  // untranslated
Blockly.Msg["UNKNOWN"] = "Unknown";  // untranslated
Blockly.Msg["UNNAMED_KEY"] = "tiada nama";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "Perkara";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Hasilkan 'set %1'";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Kembalikan nilai pemboleh ubah ini.";
Blockly.Msg["VARIABLES_SET"] = "set %1 ke %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Hasilkan 'set %1'";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Set pembolehubah ini supaya sama dengan input.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Pemboleh ubah '%1' sudah wujud.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Pemboleh ubah '%1' sudah wujud untuk jenis lain: '%2'";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "A variable named '%1' already exists as a parameter in the procedure '%2'.";  // untranslated
Blockly.Msg["WINDOWS"] = "Windows";  // untranslated
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Say something...";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
