// blockly/msg/mt.js
/* eslint-disable */
;(function(root, factory) {
  if (typeof define === 'function' && define.amd) { // AMD
    define([], factory);
  } else if (typeof exports === 'object') { // Node.js
    module.exports = factory();
  } else { // Browser
    var messages = factory();
    for (var key in messages) {
      root.Blockly.Msg[key] = messages[key];
    }
  }
}(this, function() {
// This file was automatically generated.  Do not modify.

'use strict';

var Blockly = Blockly || { Msg: Object.create(null) };

Blockly.Msg["ADD_COMMENT"] = "Żid Kumment";
Blockly.Msg["ALT_KEY"] = "Alt";
Blockly.Msg["ANNOUNCE_MOVE_AFTER"] = "Moving %1 after %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_AROUND"] = "Moving %1 around %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_BEFORE"] = "Moving %1 before %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_CANCELED"] = "Canceled movement.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_INSIDE"] = "Moving %1 inside %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_OF"] = "%1 of %2";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_TO"] = "Moving %1 to %2.";  // untranslated
Blockly.Msg["ANNOUNCE_MOVE_WORKSPACE"] = "Moving %1 on workspace.";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_ELSE_IF"] = "Add else if";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_INPUT"] = "Add input";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_LIST_ITEM"] = "Add list item";  // untranslated
Blockly.Msg["ARIA_LABEL_ADD_TEXT"] = "Add text";  // untranslated
Blockly.Msg["ARIA_LABEL_BUTTON"] = "button";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT"] = "Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_COLLAPSE"] = "Collapse Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_COMMENT_EXPAND"] = "Expand Comment";  // untranslated
Blockly.Msg["ARIA_LABEL_FIELD_ANGLE"] = "%1 degrees";  // untranslated
Blockly.Msg["ARIA_LABEL_HEADING"] = "heading";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_ELSE_IF"] = "Remove else if";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_INPUT"] = "Remove input";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_LIST_ITEM"] = "Remove list item";  // untranslated
Blockly.Msg["ARIA_LABEL_REMOVE_TEXT"] = "Remove text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_ANGLE"] = "angle";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_BITMAP"] = "pixel image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_CHECKBOX"] = "checkbox";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_COLOUR"] = "color";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DATE"] = "date";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_DROPDOWN"] = "dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_GRID"] = "grid dropdown";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_IMAGE"] = "image";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_INPUT"] = "input";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_NUMBER"] = "number";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT"] = "text";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_ARGUMENT"] = "input name";  // untranslated
Blockly.Msg["ARIA_TYPE_FIELD_TEXT_INPUT_PROCEDURE"] = "function name";  // untranslated
Blockly.Msg["BACKSPACE_KEY"] = "Backspace";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_PREFIX"] = "Begin %1";  // untranslated
Blockly.Msg["BLOCK_LABEL_BEGIN_STACK"] = "Begin stack";  // untranslated
Blockly.Msg["BLOCK_LABEL_COLLAPSED"] = "collapsed";  // untranslated
Blockly.Msg["BLOCK_LABEL_CONTAINER"] = "container";  // untranslated
Blockly.Msg["BLOCK_LABEL_DISABLED"] = "disabled";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_BRANCHES"] = "has %1 branches";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUT"] = "has input";  // untranslated
Blockly.Msg["BLOCK_LABEL_HAS_INPUTS"] = "has inputs";  // untranslated
Blockly.Msg["BLOCK_LABEL_REPLACEABLE"] = "replaceable";  // untranslated
Blockly.Msg["BLOCK_LABEL_STACK_BLOCKS"] = "%1 stack blocks";  // untranslated
Blockly.Msg["BLOCK_LABEL_STATEMENT"] = "statement";  // untranslated
Blockly.Msg["BLOCK_LABEL_TOOLBOX_CATEGORY"] = "%1 category";  // untranslated
Blockly.Msg["BLOCK_LABEL_VALUE"] = "value";  // untranslated
Blockly.Msg["BUBBLE_LABEL_COMMENT"] = "Comment: %1";  // untranslated
Blockly.Msg["BUBBLE_LABEL_DEFAULT"] = "Bubble";  // untranslated
Blockly.Msg["BUBBLE_LABEL_WARNING"] = "Warning: %1";  // untranslated
Blockly.Msg["CANNOT_DELETE_VARIABLE_PROCEDURE"] = "Varjabbli '%1' ma tħassarx għax huwa parti mit-tifsira tal-funzjoni '%2'";
Blockly.Msg["CAPS_LOCK_KEY"] = "Caps Lock";  // untranslated
Blockly.Msg["CHANGE_VALUE_TITLE"] = "Biddel valur:";
Blockly.Msg["CHROME_OS"] = "ChromeOS";
Blockly.Msg["CLEAN_UP"] = "Naddaf Blokok";
Blockly.Msg["CLOSE"] = "Agħlaq";
Blockly.Msg["COLLAPSED_WARNINGS_WARNING"] = "Blokok magħluqa fiha twissijijiet.";
Blockly.Msg["COLLAPSE_ALL"] = "Blokok Kollass";
Blockly.Msg["COLLAPSE_BLOCK"] = "Blokka Kollass";
Blockly.Msg["COLOUR_BLEND_COLOUR1"] = "lewn 1";
Blockly.Msg["COLOUR_BLEND_COLOUR2"] = "lewn 2";
Blockly.Msg["COLOUR_BLEND_HELPURL"] = "https://meyerweb.com/eric/tools/color-blend/#:::rgbp";  // untranslated
Blockly.Msg["COLOUR_BLEND_RATIO"] = "tidqis";
Blockly.Msg["COLOUR_BLEND_TITLE"] = "ħallat";
Blockly.Msg["COLOUR_BLEND_TOOLTIP"] = "Tħallaq żewġ ilwien flimkien bit-tidqis mogħti (0.0 - 1.0).";
Blockly.Msg["COLOUR_PICKER_HELPURL"] = "https://en.wikipedia.org/wiki/Color";  // untranslated
Blockly.Msg["COLOUR_PICKER_TOOLTIP"] = "Agħżel lewn mil-paletta.";
Blockly.Msg["COLOUR_RANDOM_HELPURL"] = "http://randomcolour.com";  // untranslated
Blockly.Msg["COLOUR_RANDOM_TITLE"] = "lewn addoċċ";
Blockly.Msg["COLOUR_RANDOM_TOOLTIP"] = "Agħżel kulur bl-addoċċ.";
Blockly.Msg["COLOUR_RGB_BLUE"] = "blu";
Blockly.Msg["COLOUR_RGB_GREEN"] = "aħdar";
Blockly.Msg["COLOUR_RGB_HELPURL"] = "https://www.december.com/html/spec/colorpercompact.html";  // untranslated
Blockly.Msg["COLOUR_RGB_RED"] = "aħmar";
Blockly.Msg["COLOUR_RGB_TITLE"] = "lewwen bi";
Blockly.Msg["COLOUR_RGB_TOOLTIP"] = "Oħloq kulur bl-ammont magħżul t'aħmar, aħdar, u blu. L-ammonti kollha għandhom ikunu bejn 0 u 100.";
Blockly.Msg["COMMAND_KEY"] = "⌘ Kmand";
Blockly.Msg["CONTEXT_MENU_KEY"] = "≣ Menu";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#loop-termination-blocks";  // untranslated
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK"] = "oħroġ mid-dawra";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE"] = "kompli bl-iterazzjoni tad-dawra li jmiss";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_BREAK"] = "Oħroġ mid-dawra kontenitura.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_TOOLTIP_CONTINUE"] = "Aqbeż il-bqija ta' did-dawra, u kompli bl-iterazzjoni li jmiss.";
Blockly.Msg["CONTROLS_FLOW_STATEMENTS_WARNING"] = "Twissija: Dil-blokka tista' tintuża f'dawra biss.";
Blockly.Msg["CONTROLS_FOREACH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#for-each";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_TITLE"] = "għal kull oġġett %1 fil-lista %2";
Blockly.Msg["CONTROLS_FOREACH_TOOLTIP"] = "Għal kull oġġett f'lista, tagħmel il-varjabbli '%1' għall-oġġett, u mbagħad tagħmel xi stqarrijiet.";
Blockly.Msg["CONTROLS_FOR_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#count-with";  // untranslated
Blockly.Msg["CONTROLS_FOR_TITLE"] = "għodd b'%1 minn %2 għal %3 b'%4";
Blockly.Msg["CONTROLS_FOR_TOOLTIP"] = "Agħmel lill-varjabbli '%1' tieħu l-valuri min-numru bidjien għan-numru temmiem, mgħadded mill-intervall magħżul, u agħmel il-blokok magħżula";
Blockly.Msg["CONTROLS_IF_ELSEIF_TOOLTIP"] = "Żid qagħda għall-blokka jekk";
Blockly.Msg["CONTROLS_IF_ELSE_TOOLTIP"] = "Żid qagħda finali u koprakollox għall-blokka.";
Blockly.Msg["CONTROLS_IF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/IfElse";  // untranslated
Blockly.Msg["CONTROLS_IF_IF_TOOLTIP"] = "Żid, neħħi, jew rijordna biċċiet biex tirrikonfigura dil-blokka jekk.";
Blockly.Msg["CONTROLS_IF_MSG_ELSE"] = "inkella";
Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"] = "inkella jekk";
Blockly.Msg["CONTROLS_IF_MSG_IF"] = "jekk";
Blockly.Msg["CONTROLS_IF_TOOLTIP_1"] = "Meta valur jivvera, tagħmel xi stqarrijiet.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_2"] = "Meta valur jivvera, tagħmel l-ewwel blokka ta' stqarrijiet. Inkella, tagħmel it-tieni blokka ta' stqarrijiet.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_3"] = "Meta l-ewwel valur jivvera, tagħmel l-ewwel blokka ta' stqarrijiet. Inkella, jekk it-tieni valur jivvera, tagħmel it-tieni blokka ta' stqarrijiet.";
Blockly.Msg["CONTROLS_IF_TOOLTIP_4"] = "Meta l-ewwel valur jivvera, tagħmel l-ewwel blokka ta' stqarrijiet. Inkella, jekk it-tieni valur jivvera, tagħmel l-aħħar blokka ta' stqarrijiet.";
Blockly.Msg["CONTROLS_REPEAT_HELPURL"] = "https://en.wikipedia.org/wiki/For_loop";  // untranslated
Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"] = "agħmel";
Blockly.Msg["CONTROLS_REPEAT_TITLE"] = "tenna %1 darbiet";
Blockly.Msg["CONTROLS_REPEAT_TOOLTIP"] = "Tenni stqarrijiet diversi drabi";
Blockly.Msg["CONTROLS_WHILEUNTIL_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Loops#repeat";  // untranslated
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_UNTIL"] = "tenni sakemm";
Blockly.Msg["CONTROLS_WHILEUNTIL_OPERATOR_WHILE"] = "tenni waqt";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL"] = "Meta valur jiffalza, tagħmel xi stqarrijiet.";
Blockly.Msg["CONTROLS_WHILEUNTIL_TOOLTIP_WHILE"] = "Meta valur jivvera, tagħmel xi stqarrijiet.";
Blockly.Msg["CONTROL_KEY"] = "Ktrl";
Blockly.Msg["COPY_SHORTCUT"] = "Ikkopja";
Blockly.Msg["CURRENT_BLOCK_ANNOUNCEMENT"] = "Current block: %1";  // untranslated
Blockly.Msg["CUT_SHORTCUT"] = "Aqta'";
Blockly.Msg["DELETE_ALL_BLOCKS"] = "Ħassar %1 blokok?";
Blockly.Msg["DELETE_BLOCK"] = "Ħassar Blokka";
Blockly.Msg["DELETE_KEY"] = "Delete";  // untranslated
Blockly.Msg["DELETE_VARIABLE"] = "Ħassar il-varjabbli '%1'";
Blockly.Msg["DELETE_VARIABLE_CONFIRMATION"] = "Ħassar %1 użi tal-varjabbli '%2'?";
Blockly.Msg["DELETE_X_BLOCKS"] = "Ħassar %1 Blokok";
Blockly.Msg["DIALOG_CANCEL"] = "Xejjen";
Blockly.Msg["DIALOG_OK"] = "Fhimt";
Blockly.Msg["DISABLE_BLOCK"] = "Blokka Diżattiva";
Blockly.Msg["DUPLICATE_BLOCK"] = "Irdoppja";
Blockly.Msg["DUPLICATE_COMMENT"] = "Irdoppja Kumment";
Blockly.Msg["EDIT_BLOCK_CONTENTS"] = "Biddel kontenut il-Blokka";
Blockly.Msg["ENABLE_BLOCK"] = "Blokka Attiva";
Blockly.Msg["END_KEY"] = "End";  // untranslated
Blockly.Msg["ENTER_KEY"] = "Enter";  // untranslated
Blockly.Msg["ESCAPE"] = "Escape";  // untranslated
Blockly.Msg["EXPAND_ALL"] = "Blokok Espandi";
Blockly.Msg["EXPAND_BLOCK"] = "Blokka Espandi";
Blockly.Msg["EXTERNAL_INPUTS"] = "Daħliet Esterni";
Blockly.Msg["FIELD_LABEL_CHECKBOX_CHECKED"] = "Checked";  // untranslated
Blockly.Msg["FIELD_LABEL_CHECKBOX_UNCHECKED"] = "Not checked";  // untranslated
Blockly.Msg["FIELD_LABEL_EDIT_PREFIX"] = "Edit %1";  // untranslated
Blockly.Msg["FIELD_LABEL_EMPTY"] = "empty";  // untranslated
Blockly.Msg["FIELD_LABEL_OPTION_INDEX"] = "Option %1";  // untranslated
Blockly.Msg["FIELD_LABEL_VARIABLE"] = "Variable '%1'";  // untranslated
Blockly.Msg["HELP"] = "Għajnuna";
Blockly.Msg["HELP_PROMPT"] = "Agħfas %1 għal għajnuna dwar il-kontrolli tat-tastiera";
Blockly.Msg["HOME_KEY"] = "Home";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_CLOSED"] = "Open Comment";  // untranslated
Blockly.Msg["ICON_LABEL_COMMENT_OPEN"] = "Close Comment";  // untranslated
Blockly.Msg["ICON_LABEL_DEFAULT"] = "Icon";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_CLOSED"] = "Edit this block";  // untranslated
Blockly.Msg["ICON_LABEL_MUTATOR_OPEN"] = "Close block editor";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_CLOSED"] = "Open Warning";  // untranslated
Blockly.Msg["ICON_LABEL_WARNING_OPEN"] = "Close Warning";  // untranslated
Blockly.Msg["INLINE_INPUTS"] = "Daħliet Inlinjati";
Blockly.Msg["INPUT_LABEL_CONDITION"] = "condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_A"] = "first condition";  // untranslated
Blockly.Msg["INPUT_LABEL_CONDITION_B"] = "second condition";  // untranslated
Blockly.Msg["INPUT_LABEL_EMPTY"] = "Empty";  // untranslated
Blockly.Msg["INPUT_LABEL_END_STATEMENT"] = "End %1";  // untranslated
Blockly.Msg["INPUT_LABEL_INDEX"] = "input %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_CREATE_WITH_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_DELIMITER"] = "delimiter";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_LIST_FROM_TEXT"] = "text to split";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_POSITION"] = "position within list";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_ITEM"] = "value to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_REPEAT_NUM"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TEXT_FROM_LIST"] = "list to join";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHANGE"] = "list to change";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_TO_CHECK"] = "list to check";  // untranslated
Blockly.Msg["INPUT_LABEL_LISTS_VALUE_TO_SET"] = "value to set";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_BY"] = "increment";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_FROM"] = "starting number";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_LIST"] = "list to iterate over";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TIMES"] = "number of times to repeat";  // untranslated
Blockly.Msg["INPUT_LABEL_LOOP_TO"] = "ending number";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CHANGE_BY"] = "amount to change by";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_CONSTRAIN_VALUE"] = "number to constrain";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVIDEND"] = "dividend";  // untranslated
Blockly.Msg["INPUT_LABEL_MATH_DIVISOR"] = "divisor";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER"] = "number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_A"] = "first number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_X"] = "x coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_ATAN2_Y"] = "y coordinate";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_B"] = "second number";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_LIST"] = "list of numbers";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MAX"] = "maximum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_MIN"] = "minimum";  // untranslated
Blockly.Msg["INPUT_LABEL_NUMBER_TO_CHECK"] = "number to check";  // untranslated
Blockly.Msg["INPUT_LABEL_STATEMENT"] = "statement position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_APPEND"] = "value to append";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_END_POSITION"] = "end position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_JOIN_ITEM"] = "value %1";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_POSITION"] = "letter position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_PROMPT_MESSAGE"] = "message";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_START_POSITION"] = "start position";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHANGE"] = "text to change";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_CHECK"] = "text to check";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_FIND"] = "text to find";  // untranslated
Blockly.Msg["INPUT_LABEL_TEXT_TO_REPLACE"] = "text to replace";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE"] = "value position";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_A"] = "first value";  // untranslated
Blockly.Msg["INPUT_LABEL_VALUE_B"] = "second value";  // untranslated
Blockly.Msg["INPUT_LABEL_VARIABLES_SET"] = "value to set";  // untranslated
Blockly.Msg["INSERT_KEY"] = "Insert";  // untranslated
Blockly.Msg["KEYBOARD_NAV_BLOCK_NAVIGATION_HINT"] = "Use %1 to navigate inside of blocks.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_CONSTRAINED_MOVE_HINT"] = "Uża l-vleġeġ biex tiċċaqlaq, imbagħad %1 biex taċċetta l-post";
Blockly.Msg["KEYBOARD_NAV_COPIED_HINT"] = "Ikkupjat. Agħfas %1 biex timpostja";
Blockly.Msg["KEYBOARD_NAV_CUT_HINT"] = "Maqtugħ. Agħfas %1 biex timpostja";
Blockly.Msg["KEYBOARD_NAV_FLYOUT_LABEL_HINT"] = "Use the arrow keys to navigate to a block, or press %1 to go to the next heading.";  // untranslated
Blockly.Msg["KEYBOARD_NAV_UNCONSTRAINED_MOVE_HINT"] = "Żomm %1 u uża l-vleġeġ biex tiċċaqlaq b'ħielsa, imbagħad %2 biex taċċetta l-post";
Blockly.Msg["KEYBOARD_NAV_WORKSPACE_NAVIGATION_HINT"] = "Use the arrow keys to navigate.";  // untranslated
Blockly.Msg["LINUX"] = "Linux";
Blockly.Msg["LISTS_CREATE_EMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-empty-list";  // untranslated
Blockly.Msg["LISTS_CREATE_EMPTY_TITLE"] = "oħloq lista vojta";
Blockly.Msg["LISTS_CREATE_EMPTY_TOOLTIP"] = "Tiġi b'lista, ta' tul 0, li ma fiha l-ebda rikordju tad-data";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TITLE_ADD"] = "lista";
Blockly.Msg["LISTS_CREATE_WITH_CONTAINER_TOOLTIP"] = "Żid, neħħi, jew rijordna biċċiet biex tirrikonfigura dil-blokka lista.";
Blockly.Msg["LISTS_CREATE_WITH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_CREATE_WITH_INPUT_WITH"] = "oħloq test bi";
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TOOLTIP"] = "Żid oġġett mal-lista.";
Blockly.Msg["LISTS_CREATE_WITH_TOOLTIP"] = "Oħloq lista b'numru t'oġġetti.";
Blockly.Msg["LISTS_GET_INDEX_FIRST"] = "l-ewwel";
Blockly.Msg["LISTS_GET_INDEX_FROM_END"] = "# mill-aħħar";
Blockly.Msg["LISTS_GET_INDEX_FROM_START"] = "#";
Blockly.Msg["LISTS_GET_INDEX_GET"] = "ikseb";
Blockly.Msg["LISTS_GET_INDEX_GET_REMOVE"] = "ikseb u neħħi";
Blockly.Msg["LISTS_GET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-items-from-a-list";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_LAST"] = "l-aħħar";
Blockly.Msg["LISTS_GET_INDEX_RANDOM"] = "addoċċ";
Blockly.Msg["LISTS_GET_INDEX_REMOVE"] = "neħħi";
Blockly.Msg["LISTS_GET_INDEX_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FIRST"] = "Tiġi bl-ewwel oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_FROM"] = "Tiġi bl-oġġett fil-post magħżul f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_LAST"] = "Tiġi bl-aħħar oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_RANDOM"] = "Tiġi b'oġġett addoċċ f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FIRST"] = "Tneħħi u tiġi bl-ewwel oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_FROM"] = "Tneħħi u tiġi bl-oġġett fil-post magħżul f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_LAST"] = "Tneħħi u tiġi bl-aħħar oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_GET_REMOVE_RANDOM"] = "Tneħħi u tiġi b'oġġett addoċċ f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FIRST"] = "Tneħħi l-ewwel oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_FROM"] = "Tneħħi l-oġġett fil-post magħżul f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_LAST"] = "Tneħħi l-aħħar oġġett f'lista.";
Blockly.Msg["LISTS_GET_INDEX_TOOLTIP_REMOVE_RANDOM"] = "Tneħħi oġġett addoċċ f'lista.";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_END"] = "għal # mill-aħħar";
Blockly.Msg["LISTS_GET_SUBLIST_END_FROM_START"] = "għal #";
Blockly.Msg["LISTS_GET_SUBLIST_END_LAST"] = "għall-aħħar";
Blockly.Msg["LISTS_GET_SUBLIST_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#getting-a-sublist";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_START_FIRST"] = "ġib sottolista mill-ewwel";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_END"] = "ġib sottolista minn # mill-aħħar";
Blockly.Msg["LISTS_GET_SUBLIST_START_FROM_START"] = "ġib sottolista minn #";
Blockly.Msg["LISTS_GET_SUBLIST_TAIL"] = "";  // untranslated
Blockly.Msg["LISTS_GET_SUBLIST_TOOLTIP"] = "Toħloq kopja tal-biċċa magħżula ta' lista.";
Blockly.Msg["LISTS_INDEX_FROM_END_TOOLTIP"] = "%1 hi l-aħħar oġġett.";
Blockly.Msg["LISTS_INDEX_FROM_START_TOOLTIP"] = "%1 hi l-ewwel oġġett.";
Blockly.Msg["LISTS_INDEX_OF_FIRST"] = "sib l-ewwel ħarġa tal-oġġett";
Blockly.Msg["LISTS_INDEX_OF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#finding-items-in-a-list";  // untranslated
Blockly.Msg["LISTS_INDEX_OF_LAST"] = "sib l-aħħar ħarġa tal-oġġett";
Blockly.Msg["LISTS_INDEX_OF_TOOLTIP"] = "Tiġi b'werrej tal-ewwel/aħħar ħarġa tal-oġġett fil-lista. Tiġi b'%1 jekk l-oġġett ma jinstabx.";
Blockly.Msg["LISTS_INLIST"] = "fil-lista";
Blockly.Msg["LISTS_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#is-empty";  // untranslated
Blockly.Msg["LISTS_ISEMPTY_TITLE"] = "%1 hi vojta";
Blockly.Msg["LISTS_ISEMPTY_TOOLTIP"] = "Tiġi b'vera jekk il-lista hi vojta.";
Blockly.Msg["LISTS_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#length-of";  // untranslated
Blockly.Msg["LISTS_LENGTH_TITLE"] = "tul ta' %1";
Blockly.Msg["LISTS_LENGTH_TOOLTIP"] = "Tiġi bit-tul tal-lista";
Blockly.Msg["LISTS_REPEAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#create-list-with";  // untranslated
Blockly.Msg["LISTS_REPEAT_TITLE"] = "oħloq lista bl-oġġett %1 mtenni %2 darbiet";
Blockly.Msg["LISTS_REPEAT_TOOLTIP"] = "Toħloq lista li fiha l-valur mogħti mtenni għan-numru ta' drabi magħżul.";
Blockly.Msg["LISTS_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#reversing-a-list";  // untranslated
Blockly.Msg["LISTS_REVERSE_MESSAGE0"] = "aqbleb %1";
Blockly.Msg["LISTS_REVERSE_TOOLTIP"] = "Aqleb kopja ta' lista.";
Blockly.Msg["LISTS_SET_INDEX_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#in-list--set";  // untranslated
Blockly.Msg["LISTS_SET_INDEX_INPUT_TO"] = "bħala";
Blockly.Msg["LISTS_SET_INDEX_INSERT"] = "daħħal fi";
Blockly.Msg["LISTS_SET_INDEX_SET"] = "irranġa";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FIRST"] = "Iddaħħal l-oġġett fil-bidu ta' lista.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_FROM"] = "Iddaħħal l-oġġett fil-post magħżul f'lista";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_LAST"] = "Iżżid l-oġġett fl-aħħar ta' lista.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_INSERT_RANDOM"] = "Iddaħħal l-oġġett bl-addoċċ f'lista.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FIRST"] = "Tirranġa l-ewwel oġġett f'lista.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_FROM"] = "Tirranġa l-oġġett fil-post magħżul f'lista";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_LAST"] = "Tirranġa l-aħħar oġġett f'lista.";
Blockly.Msg["LISTS_SET_INDEX_TOOLTIP_SET_RANDOM"] = "Tirranġa oġġett addoċċ f'lista.";
Blockly.Msg["LISTS_SORT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#sorting-a-list";  // untranslated
Blockly.Msg["LISTS_SORT_ORDER_ASCENDING"] = "telgħana";
Blockly.Msg["LISTS_SORT_ORDER_DESCENDING"] = "nżula";
Blockly.Msg["LISTS_SORT_TITLE"] = "ordna %1 %2 %3";
Blockly.Msg["LISTS_SORT_TOOLTIP"] = "Ordna kopja ta' lista.";
Blockly.Msg["LISTS_SORT_TYPE_IGNORECASE"] = "alfabetiku, injora kapitali";
Blockly.Msg["LISTS_SORT_TYPE_NUMERIC"] = "numweiku";
Blockly.Msg["LISTS_SORT_TYPE_TEXT"] = "alfabetiku";
Blockly.Msg["LISTS_SPLIT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Lists#splitting-strings-and-joining-lists";  // untranslated
Blockly.Msg["LISTS_SPLIT_LIST_FROM_TEXT"] = "agħmel lista mit-test";
Blockly.Msg["LISTS_SPLIT_TEXT_FROM_LIST"] = "agħmel test mil-lista";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_JOIN"] = "Għaqqad lista ta' testi f'test wieħed, imfarrda minn delimitar.";
Blockly.Msg["LISTS_SPLIT_TOOLTIP_SPLIT"] = "Aqsam test f'lista ta' testi, b'kisra ma' kull delimitatur.";
Blockly.Msg["LISTS_SPLIT_WITH_DELIMITER"] = "bid-delimitatur";
Blockly.Msg["LOGIC_BOOLEAN_FALSE"] = "falz";
Blockly.Msg["LOGIC_BOOLEAN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#values";  // untranslated
Blockly.Msg["LOGIC_BOOLEAN_TOOLTIP"] = "Tiġi b'veru jew falz.";
Blockly.Msg["LOGIC_BOOLEAN_TRUE"] = "vera";
Blockly.Msg["LOGIC_COMPARE_EQ_ARIA"] = "equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GTE_ARIA"] = "greater than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_GT_ARIA"] = "greater than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_HELPURL"] = "https://en.wikipedia.org/wiki/Inequality_(mathematics)";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LTE_ARIA"] = "less than or equal to";  // untranslated
Blockly.Msg["LOGIC_COMPARE_LT_ARIA"] = "less than";  // untranslated
Blockly.Msg["LOGIC_COMPARE_NEQ_ARIA"] = "not equals";  // untranslated
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_EQ"] = "Iġi b'veru jekk iż-żewġ daħliet huma ndaqs.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GT"] = "Iġi b'veru jekk l-ewwel daħla hija ikbar mit-tieni daħla.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_GTE"] = "Iġi b'veru jekk l-ewwel daħla hija ikbar jew indaqs għat-tieni daħla.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LT"] = "Iġi b'veru jekk l-ewwel daħla hija iżgħar mit-tieni daħla.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_LTE"] = "Iġi b'veru jekk l-ewwel daħla hija iżgħar jew indaqs għat-tieni daħla.";
Blockly.Msg["LOGIC_COMPARE_TOOLTIP_NEQ"] = "Iġi b'veru jekk iż-żewġ daħliet mhumiex indaqs.";
Blockly.Msg["LOGIC_NEGATE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#not";  // untranslated
Blockly.Msg["LOGIC_NEGATE_TITLE"] = "mhux %1";
Blockly.Msg["LOGIC_NEGATE_TOOLTIP"] = "Tiġi b'veru jekk id-daħla hija vera. Tiġi b'falz jekk id-daħla hija falza.";
Blockly.Msg["LOGIC_NULL"] = "xejn";
Blockly.Msg["LOGIC_NULL_HELPURL"] = "https://en.wikipedia.org/wiki/Nullable_type";  // untranslated
Blockly.Msg["LOGIC_NULL_TOOLTIP"] = "Tiġi b'xejn.";
Blockly.Msg["LOGIC_OPERATION_AND"] = "u";
Blockly.Msg["LOGIC_OPERATION_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Logic#logical-operations";  // untranslated
Blockly.Msg["LOGIC_OPERATION_OR"] = "jew";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_AND"] = "Iġi b'veru jekk iż-żewġ daħliet huma veru.";
Blockly.Msg["LOGIC_OPERATION_TOOLTIP_OR"] = "Iġi b'veru jekk mill-inqas waħda mid-daħliet huma veru.";
Blockly.Msg["LOGIC_TERNARY_CONDITION"] = "test";
Blockly.Msg["LOGIC_TERNARY_HELPURL"] = "https://en.wikipedia.org/wiki/%3F:";  // untranslated
Blockly.Msg["LOGIC_TERNARY_IF_FALSE"] = "jekk falza";
Blockly.Msg["LOGIC_TERNARY_IF_TRUE"] = "jekk vera";
Blockly.Msg["LOGIC_TERNARY_TOOLTIP"] = "Ara l-qagħda f'test. Jekk il-qagħda hija vera, tiġi bil-valura 'jekk vera'; inkella tiġi bil-valura 'jekk falza'.";
Blockly.Msg["MAC_OS"] = "macOS";
Blockly.Msg["MATH_ADDITION_SYMBOL"] = "+";  // untranslated
Blockly.Msg["MATH_ADDITION_SYMBOL_ARIA"] = "plus";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_HELPURL"] = "https://en.wikipedia.org/wiki/Arithmetic";  // untranslated
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_ADD"] = "Iġi bis-somma taż-żewġ numri.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_DIVIDE"] = "Iġi bil-kwozjent taż-żewġ numri.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MINUS"] = "Iġi bid-differenza taż-żewġ numri.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_MULTIPLY"] = "Iġi bil-prodott taż-żewġ numri.";
Blockly.Msg["MATH_ARITHMETIC_TOOLTIP_POWER"] = "Iġi bl-ewwel numri mgħolli għall-qawwa tat-tieni numri.";
Blockly.Msg["MATH_ATAN2_HELPURL"] = "https://en.wikipedia.org/wiki/Atan2";  // untranslated
Blockly.Msg["MATH_ATAN2_TITLE"] = "atan2 ta' X:%1 Y:%2";
Blockly.Msg["MATH_ATAN2_TOOLTIP"] = "Iġi bl-arktanġent tal-punt (X, Y) f'gradi minn -180 sa 180.";
Blockly.Msg["MATH_CHANGE_HELPURL"] = "https://en.wikipedia.org/wiki/Programming_idiom#Incrementing_a_counter";  // untranslated
Blockly.Msg["MATH_CHANGE_TITLE"] = "ibdel %1 b'%2";
Blockly.Msg["MATH_CHANGE_TOOLTIP"] = "Żid numru għall-varjabbli '%1'.";
Blockly.Msg["MATH_CONSTANT_E_ARIA"] = "e";  // untranslated
Blockly.Msg["MATH_CONSTANT_GOLDEN_RATIO_ARIA"] = "golden ratio";  // untranslated
Blockly.Msg["MATH_CONSTANT_HELPURL"] = "https://en.wikipedia.org/wiki/Mathematical_constant";  // untranslated
Blockly.Msg["MATH_CONSTANT_INFINITY_ARIA"] = "infinity";  // untranslated
Blockly.Msg["MATH_CONSTANT_PI_ARIA"] = "pi";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT1_2_ARIA"] = "square root of 1 over 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_SQRT2_ARIA"] = "square root of 2";  // untranslated
Blockly.Msg["MATH_CONSTANT_TOOLTIP"] = "Iġi b'wieħed mill-kostanti komuni: π (3.141…), e (2.718…), φ (1.618…), rqkwd(2) (1.414…), rqkwd(½) (0.707…), jew ∞ (infinità).";
Blockly.Msg["MATH_CONSTRAIN_HELPURL"] = "https://en.wikipedia.org/wiki/Clamping_(graphics)";  // untranslated
Blockly.Msg["MATH_CONSTRAIN_TITLE"] = "Ġiegħel lil %1 baxx %2 għoli %3";
Blockly.Msg["MATH_CONSTRAIN_TOOLTIP"] = "Iġġiegħel numru biex tkun fil-limiti magħżula (inklużiva).";
Blockly.Msg["MATH_DIVISION_SYMBOL"] = "÷";  // untranslated
Blockly.Msg["MATH_DIVISION_SYMBOL_ARIA"] = "divided by";  // untranslated
Blockly.Msg["MATH_IS_DIVISIBLE_BY"] = "jinqasam fi";
Blockly.Msg["MATH_IS_EVEN"] = "hu żewġ";
Blockly.Msg["MATH_IS_NEGATIVE"] = "hu negattiv";
Blockly.Msg["MATH_IS_ODD"] = "hu fard";
Blockly.Msg["MATH_IS_POSITIVE"] = "hu pożittiv";
Blockly.Msg["MATH_IS_PRIME"] = "hu prim";
Blockly.Msg["MATH_IS_TOOLTIP"] = "Ara jekk numru huwa żewġ, fard, prim, sħiħ, pożittiv, negattiv, jew jekk tinqasam minn ċertu numru. Tiġi b'veru jew falz.";
Blockly.Msg["MATH_IS_WHOLE"] = "hu sħiħ";
Blockly.Msg["MATH_MODULO_HELPURL"] = "https://en.wikipedia.org/wiki/Modulo_operation";  // untranslated
Blockly.Msg["MATH_MODULO_TITLE"] = "bqija ta' %1 ÷ %2";
Blockly.Msg["MATH_MODULO_TOOLTIP"] = "Iġi bil-bqija mill-qsim taż-żewġ numri.";
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL"] = "×";  // untranslated
Blockly.Msg["MATH_MULTIPLICATION_SYMBOL_ARIA"] = "times";  // untranslated
Blockly.Msg["MATH_NUMBER_HELPURL"] = "https://en.wikipedia.org/wiki/Number";  // untranslated
Blockly.Msg["MATH_NUMBER_TOOLTIP"] = "Numru.";
Blockly.Msg["MATH_ONLIST_HELPURL"] = "";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_AVERAGE"] = "medja tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX"] = "mas tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_MAX_ARIA"] = "maximum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MEDIAN"] = "medja tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN"] = "min tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_MIN_ARIA"] = "minimum";  // untranslated
Blockly.Msg["MATH_ONLIST_OPERATOR_MODE"] = "modi tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_RANDOM"] = "oġġett addoċċ tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_STD_DEV"] = "dejvar standard tal-lista";
Blockly.Msg["MATH_ONLIST_OPERATOR_SUM"] = "somma tal-lista";
Blockly.Msg["MATH_ONLIST_TOOLTIP_AVERAGE"] = "Iġi bil-medja (medju aritmetiku) tal-valuri numeriċi fil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MAX"] = "Iġi bl-ikbar numru fil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MEDIAN"] = "Iġi bin-numru medjan fil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MIN"] = "Iġi bl-iżgħar numru fil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_MODE"] = "Iġi bil-lista tal-ikmen oġġett(i) fil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_RANDOM"] = "Iġi b'element addoċċ mil-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_STD_DEV"] = "Iġi bid-devjar standard tal-lista.";
Blockly.Msg["MATH_ONLIST_TOOLTIP_SUM"] = "Iġi bis-somma tan-numri kollha fil-lista.";
Blockly.Msg["MATH_POWER_SYMBOL"] = "^";  // untranslated
Blockly.Msg["MATH_POWER_SYMBOL_ARIA"] = "to the power of";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_FLOAT_TITLE_RANDOM"] = "qasma addoċċa";
Blockly.Msg["MATH_RANDOM_FLOAT_TOOLTIP"] = "Iġi b'qasma addoċċa bejn 0.0 (inklużiv) u 1.0 (esklużiv).";
Blockly.Msg["MATH_RANDOM_INT_HELPURL"] = "https://en.wikipedia.org/wiki/Random_number_generation";  // untranslated
Blockly.Msg["MATH_RANDOM_INT_TITLE"] = "Sħiħa addoċċa minn %1 sa %2";
Blockly.Msg["MATH_RANDOM_INT_TOOLTIP"] = "Iġi b'sħiħa addoċċa bejn żewġ limiti magħżula, inkużiva.";
Blockly.Msg["MATH_ROUND_HELPURL"] = "https://en.wikipedia.org/wiki/Rounding";  // untranslated
Blockly.Msg["MATH_ROUND_OPERATOR_ROUND"] = "qarrab";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDDOWN"] = "qarrab 'il isfel";
Blockly.Msg["MATH_ROUND_OPERATOR_ROUNDUP"] = "qarrab 'il fuq";
Blockly.Msg["MATH_ROUND_TOOLTIP"] = "Qarrab numru 'l fuq jew 'l isfel.";
Blockly.Msg["MATH_SINGLE_HELPURL"] = "https://en.wikipedia.org/wiki/Square_root";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE"] = "assolut";
Blockly.Msg["MATH_SINGLE_OP_ABSOLUTE_ARIA"] = "absolute value";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_EXP_ARIA"] = "e to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LN_ARIA"] = "natural logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_LOG10_ARIA"] = "base 10 logarithm";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_NEG_ARIA"] = "negative";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_POW10_ARIA"] = "10 to the power of";  // untranslated
Blockly.Msg["MATH_SINGLE_OP_ROOT"] = "għerq kwadru";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ABS"] = "Iġi bil-valur assolut ta' numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_EXP"] = "Iġi b'e għall-qawwa ta' numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LN"] = "Iġi bil-logaritmu naturali ta' numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_LOG10"] = "Iġi bil-logaritmu bażi 10 ta' numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_NEG"] = "Iġi bin-negattiv tan-numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_POW10"] = "Iġi b'10 għall-qawwa ta' numru.";
Blockly.Msg["MATH_SINGLE_TOOLTIP_ROOT"] = "Iġi bl-għerq kwadru ta' numru.";
Blockly.Msg["MATH_SUBTRACTION_SYMBOL"] = "-";  // untranslated
Blockly.Msg["MATH_SUBTRACTION_SYMBOL_ARIA"] = "minus";  // untranslated
Blockly.Msg["MATH_TRIG_ACOS"] = "akos";
Blockly.Msg["MATH_TRIG_ACOS_ARIA"] = "inverse cosine";  // untranslated
Blockly.Msg["MATH_TRIG_ASIN"] = "asin";
Blockly.Msg["MATH_TRIG_ASIN_ARIA"] = "inverse sine";  // untranslated
Blockly.Msg["MATH_TRIG_ATAN"] = "atan";
Blockly.Msg["MATH_TRIG_ATAN_ARIA"] = "inverse tangent";  // untranslated
Blockly.Msg["MATH_TRIG_COS"] = "kos";
Blockly.Msg["MATH_TRIG_COS_ARIA"] = "cosine";  // untranslated
Blockly.Msg["MATH_TRIG_HELPURL"] = "https://en.wikipedia.org/wiki/Trigonometric_functions";  // untranslated
Blockly.Msg["MATH_TRIG_SIN"] = "sin";
Blockly.Msg["MATH_TRIG_SIN_ARIA"] = "sine";  // untranslated
Blockly.Msg["MATH_TRIG_TAN"] = "tan";
Blockly.Msg["MATH_TRIG_TAN_ARIA"] = "tangent";  // untranslated
Blockly.Msg["MATH_TRIG_TOOLTIP_ACOS"] = "Iġi bl-arkosinju ta' numru.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ASIN"] = "Iġi bl-arksinju ta' numru.";
Blockly.Msg["MATH_TRIG_TOOLTIP_ATAN"] = "Iġi bl-arktanġent ta' numru.";
Blockly.Msg["MATH_TRIG_TOOLTIP_COS"] = "Iġi bil-kosinju ta' grad (mhux radjan).";
Blockly.Msg["MATH_TRIG_TOOLTIP_SIN"] = "Iġi bis-sinju ta' grad (mhux radjan).";
Blockly.Msg["MATH_TRIG_TOOLTIP_TAN"] = "Iġi bit-tanġent ta' grad (mhux radjan).";
Blockly.Msg["MOVE_BLOCK"] = "Mexxi Blokka";
Blockly.Msg["NEW_COLOUR_VARIABLE"] = "Oħloq varjabbli lewni...";
Blockly.Msg["NEW_NUMBER_VARIABLE"] = "Oħloq varjabbli numeriku...";
Blockly.Msg["NEW_STRING_VARIABLE"] = "Oħloq varjabbli spaginu...";
Blockly.Msg["NEW_VARIABLE"] = "Oħloq varjabbli...";
Blockly.Msg["NEW_VARIABLE_TITLE"] = "Isem il-varjabbli ġdid:";
Blockly.Msg["NEW_VARIABLE_TYPE_TITLE"] = "Tip varjabbli ġdid:";
Blockly.Msg["NO_PARENT_ANNOUNCEMENT"] = "Current block has no parent";  // untranslated
Blockly.Msg["OPEN_TRASH"] = "Open trash";  // untranslated
Blockly.Msg["OPTION_KEY"] = "⌥ Għażla";
Blockly.Msg["ORDINAL_NUMBER_SUFFIX"] = "";  // untranslated
Blockly.Msg["PAGE_DOWN_KEY"] = "Page Down";  // untranslated
Blockly.Msg["PAGE_UP_KEY"] = "Page Up";  // untranslated
Blockly.Msg["PARENT_BLOCKS_ANNOUNCEMENT"] = "Parent blocks: %1";  // untranslated
Blockly.Msg["PASTE_SHORTCUT"] = "Impostja";
Blockly.Msg["PAUSE_KEY"] = "Pause";  // untranslated
Blockly.Msg["PROCEDURES_ALLOW_STATEMENTS"] = "ħalli stqarriji";
Blockly.Msg["PROCEDURES_BEFORE_PARAMS"] = "bi:";
Blockly.Msg["PROCEDURES_CALLNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLNORETURN_TOOLTIP"] = "Ħarreġ l-funzjoni-utenta '%1'.";
Blockly.Msg["PROCEDURES_CALLRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_CALLRETURN_TOOLTIP"] = "Ħarreġ l-funzjoni-utenta '%1' u uża l-ħarġa tagħha.";
Blockly.Msg["PROCEDURES_CALL_BEFORE_PARAMS"] = "bi:";
Blockly.Msg["PROCEDURES_CALL_DISABLED_DEF_WARNING"] = "Ma tistax tħaddem il-funzjoni-utenta '%1' għax il-blokka tat-tifsira hija mitfija.";
Blockly.Msg["PROCEDURES_CREATE_DO"] = "Oħloq '%1'";
Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"] = "Fisser dil-funzjoni...";
Blockly.Msg["PROCEDURES_DEFNORETURN_DO"] = "";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"] = "agħmel xi ħaġa";
Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"] = "għal";
Blockly.Msg["PROCEDURES_DEFNORETURN_TOOLTIP"] = "Toħloq funzjoni bla ħarġa.";
Blockly.Msg["PROCEDURES_DEFRETURN_HELPURL"] = "https://en.wikipedia.org/wiki/Subroutine";  // untranslated
Blockly.Msg["PROCEDURES_DEFRETURN_RETURN"] = "ġib";
Blockly.Msg["PROCEDURES_DEFRETURN_TOOLTIP"] = "Toħloq funzjoni b'ħarġa.";
Blockly.Msg["PROCEDURES_DEF_DUPLICATE_WARNING"] = "Twissija: Dil-funzjoni għandha parametri duplikati.";
Blockly.Msg["PROCEDURES_HIGHLIGHT_DEF"] = "Lewwen tifsira tal-funzjoni";
Blockly.Msg["PROCEDURES_IFRETURN_HELPURL"] = "https://c2.com/cgi/wiki?GuardClause";  // untranslated
Blockly.Msg["PROCEDURES_IFRETURN_TOOLTIP"] = "Jekk valur huwa veru, iġi b'valur ieħor.";
Blockly.Msg["PROCEDURES_IFRETURN_WARNING"] = "Twissija: Dil-blokka tista' tintuża biss f'tifsira ta' funzjoni.";
Blockly.Msg["PROCEDURES_MUTATORARG_TITLE"] = "isem id-daħla:";
Blockly.Msg["PROCEDURES_MUTATORARG_TOOLTIP"] = "Żid daħla għall-funzjoni.";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TITLE"] = "daħliet";
Blockly.Msg["PROCEDURES_MUTATORCONTAINER_TOOLTIP"] = "Żid, neħħi, jew rijordna d-daħliet għal dil-funzjoni.";
Blockly.Msg["REDO"] = "Reġġa'";
Blockly.Msg["REMOVE_COMMENT"] = "Neħħi Kumment";
Blockly.Msg["RENAME_VARIABLE"] = "Biddel isem il-varjabbli...";
Blockly.Msg["RENAME_VARIABLE_TITLE"] = "Semmi varjabbli '%1' għal:";
Blockly.Msg["RESET_ZOOM"] = "Reset zoom";  // untranslated
Blockly.Msg["SCREENREADER_HINT"] = "Use the arrow keys to navigate. Press %1 to toggle screenreader accessibility mode.";  // untranslated
Blockly.Msg["SCREENREADER_MODE_DISABLED"] = "Screenreader mode is off, press %1 to turn it on";  // untranslated
Blockly.Msg["SCREENREADER_MODE_ENABLED"] = "Screenreader mode is on, press %1 to turn it off";  // untranslated
Blockly.Msg["SHIFT_KEY"] = "Shift";  // untranslated
Blockly.Msg["SHORTCUTS_ABORT_MOVE"] = "Abort move";  // untranslated
Blockly.Msg["SHORTCUTS_CLEANUP"] = "Clean up workspace";  // untranslated
Blockly.Msg["SHORTCUTS_CODE_NAVIGATION"] = "Tmexxija tal-kodiċi";
Blockly.Msg["SHORTCUTS_DELETE"] = "Delete";  // untranslated
Blockly.Msg["SHORTCUTS_DISCONNECT"] = "Disconnect block";  // untranslated
Blockly.Msg["SHORTCUTS_DUPLICATE"] = "Duplicate";  // untranslated
Blockly.Msg["SHORTCUTS_EDITING"] = "Tibdil";
Blockly.Msg["SHORTCUTS_ESCAPE"] = "Exit";  // untranslated
Blockly.Msg["SHORTCUTS_EXTENDED_INFORMATION"] = "Announce detailed information";  // untranslated
Blockly.Msg["SHORTCUTS_FINISH_MOVE"] = "Finish move";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_TOOLBOX"] = "Focus toolbox";  // untranslated
Blockly.Msg["SHORTCUTS_FOCUS_WORKSPACE"] = "Focus workspace";  // untranslated
Blockly.Msg["SHORTCUTS_GENERAL"] = "Ġenerali";
Blockly.Msg["SHORTCUTS_INFORMATION"] = "Announce information";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_DOWN"] = "Move down";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_LEFT"] = "Move left";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_RIGHT"] = "Move right";  // untranslated
Blockly.Msg["SHORTCUTS_MOVE_UP"] = "Move up";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_HEADING"] = "Next heading";  // untranslated
Blockly.Msg["SHORTCUTS_NEXT_STACK"] = "Next stack";  // untranslated
Blockly.Msg["SHORTCUTS_PERFORM_ACTION"] = "Edit or confirm";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_HEADING"] = "Previous heading";  // untranslated
Blockly.Msg["SHORTCUTS_PREVIOUS_STACK"] = "Previous stack";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_CONTEXT_MENU"] = "Show menu";  // untranslated
Blockly.Msg["SHORTCUTS_SHOW_TOOLTIP"] = "Show tooltip";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE"] = "Start move";  // untranslated
Blockly.Msg["SHORTCUTS_START_MOVE_STACK"] = "Start move stack";  // untranslated
Blockly.Msg["SHORTCUTS_TOGGLE_SCREENREADER_MODE"] = "Toggle screenreader mode";  // untranslated
Blockly.Msg["SPACE_KEY"] = "Space";  // untranslated
Blockly.Msg["TAB_KEY"] = "Tab";  // untranslated
Blockly.Msg["TEXT_APPEND_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_APPEND_TITLE"] = "għal %1 żid %2";
Blockly.Msg["TEXT_APPEND_TOOLTIP"] = "Żid xi test mal-varjabbli '%1'.";
Blockly.Msg["TEXT_CHANGECASE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#adjusting-text-case";  // untranslated
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_LOWERCASE"] = "għal ittri żgħar";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_TITLECASE"] = "għal Kliem Titlu";
Blockly.Msg["TEXT_CHANGECASE_OPERATOR_UPPERCASE"] = "għal ITTRI KBAR";
Blockly.Msg["TEXT_CHANGECASE_TOOLTIP"] = "Iġi b'kopja tat-test f'każ differenti";
Blockly.Msg["TEXT_CHARAT_FIRST"] = "ikseb l-ewwel ittra";
Blockly.Msg["TEXT_CHARAT_FROM_END"] = "ikseb l-ittra # mill-aħħar";
Blockly.Msg["TEXT_CHARAT_FROM_START"] = "ikseb ittra #";
Blockly.Msg["TEXT_CHARAT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-text";  // untranslated
Blockly.Msg["TEXT_CHARAT_LAST"] = "ikseb l-aħħar ittra";
Blockly.Msg["TEXT_CHARAT_RANDOM"] = "ikseb ittra addoċċa";
Blockly.Msg["TEXT_CHARAT_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_CHARAT_TITLE"] = "fit-test %1 %2";
Blockly.Msg["TEXT_CHARAT_TOOLTIP"] = "Tiġi bl-ittra fil-post magħżul.";
Blockly.Msg["TEXT_COUNT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#counting-substrings";  // untranslated
Blockly.Msg["TEXT_COUNT_MESSAGE0"] = "għodd %1 f'%2";
Blockly.Msg["TEXT_COUNT_TOOLTIP"] = "Għodd kemm-il darba xi test jidher f'xi test ieħor.";
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TOOLTIP"] = "Żid oġġett mat-test.";
Blockly.Msg["TEXT_CREATE_JOIN_TITLE_JOIN"] = "għaqqad";
Blockly.Msg["TEXT_CREATE_JOIN_TOOLTIP"] = "Żid, neħħi, jew rijordna biċċiet biex tirrikonfigura dil-blokka test.";
Blockly.Msg["TEXT_FROM_END_ARIA"] = "letter number from end";  // untranslated
Blockly.Msg["TEXT_FROM_START_ARIA"] = "letter number";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_END"] = "għall-ittra # mill-aħħar";
Blockly.Msg["TEXT_GET_SUBSTRING_END_FROM_START"] = "għall-ittra #";
Blockly.Msg["TEXT_GET_SUBSTRING_END_LAST"] = "għall-aħħar ittra";
Blockly.Msg["TEXT_GET_SUBSTRING_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#extracting-a-region-of-text";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_INPUT_IN_TEXT"] = "fit-test";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FIRST"] = "ikseb sottospaga mill-ewwel ittra";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_END"] = "ikseb sottospaga minn # ittra mill-aħħar";
Blockly.Msg["TEXT_GET_SUBSTRING_START_FROM_START"] = "ikseb sottospaga mill-ittra #";
Blockly.Msg["TEXT_GET_SUBSTRING_TAIL"] = "";  // untranslated
Blockly.Msg["TEXT_GET_SUBSTRING_TOOLTIP"] = "Tiġi bil-biċċa tat-test magħżula.";
Blockly.Msg["TEXT_INDEXOF_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#finding-text";  // untranslated
Blockly.Msg["TEXT_INDEXOF_OPERATOR_FIRST"] = "sib l-ewwel ħarġa tat-test";
Blockly.Msg["TEXT_INDEXOF_OPERATOR_LAST"] = "sib l-aħħar ħarġa tat-test";
Blockly.Msg["TEXT_INDEXOF_TITLE"] = "fit-test %1 %2 %3";
Blockly.Msg["TEXT_INDEXOF_TOOLTIP"] = "Tiġi b'werrej tal-aħħar/ewwel ħarġa tal-ewwel test fit-tieni test. Tiġi b'%1 jekk it-test ma jinstabx.";
Blockly.Msg["TEXT_ISEMPTY_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#checking-for-empty-text";  // untranslated
Blockly.Msg["TEXT_ISEMPTY_TITLE"] = "%1 hi vojta";
Blockly.Msg["TEXT_ISEMPTY_TOOLTIP"] = "Tiġi b'vera jekk it-test provdut hu vojt.";
Blockly.Msg["TEXT_JOIN_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-creation";  // untranslated
Blockly.Msg["TEXT_JOIN_TITLE_CREATEWITH"] = "oħloq test bi";
Blockly.Msg["TEXT_JOIN_TOOLTIP"] = "Oħloq biċċa test billi tgħaqqad xi numru t'elementi.";
Blockly.Msg["TEXT_LENGTH_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#text-modification";  // untranslated
Blockly.Msg["TEXT_LENGTH_TITLE"] = "tul ta' %1";
Blockly.Msg["TEXT_LENGTH_TOOLTIP"] = "Tiġi bin-numru t'ittri (ikluż l-ispazji) fit-test provdut.";
Blockly.Msg["TEXT_PRINT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#printing-text";  // untranslated
Blockly.Msg["TEXT_PRINT_TITLE"] = "stampa %1";
Blockly.Msg["TEXT_PRINT_TOOLTIP"] = "Stampa it-test, numru jew valur ieħor magħżul.";
Blockly.Msg["TEXT_PROMPT_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#getting-input-from-the-user";  // untranslated
Blockly.Msg["TEXT_PROMPT_TOOLTIP_NUMBER"] = "Staqsi lill-utent għal numru.";
Blockly.Msg["TEXT_PROMPT_TOOLTIP_TEXT"] = "Staqsi lill-utent għal xi test.";
Blockly.Msg["TEXT_PROMPT_TYPE_NUMBER"] = "staqsi għal numru b'messaġġ";
Blockly.Msg["TEXT_PROMPT_TYPE_TEXT"] = "staqsi għal test b'messaġġ";
Blockly.Msg["TEXT_REPLACE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#replacing-substrings";  // untranslated
Blockly.Msg["TEXT_REPLACE_MESSAGE0"] = "biddel %1 b'%2 f'%3";
Blockly.Msg["TEXT_REPLACE_TOOLTIP"] = "Biddel kull dehra ta' xi test b'test ieħor.";
Blockly.Msg["TEXT_REVERSE_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#reversing-text";  // untranslated
Blockly.Msg["TEXT_REVERSE_MESSAGE0"] = "aqbleb %1";
Blockly.Msg["TEXT_REVERSE_TOOLTIP"] = "Taqleb l-ordni tal-karattri fit-test.";
Blockly.Msg["TEXT_TEXT_HELPURL"] = "https://en.wikipedia.org/wiki/String_(computer_science)";  // untranslated
Blockly.Msg["TEXT_TEXT_TOOLTIP"] = "Ittra, kelma, jew linja ta' test.*";
Blockly.Msg["TEXT_TRIM_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Text#trimming-removing-spaces";  // untranslated
Blockly.Msg["TEXT_TRIM_OPERATOR_BOTH"] = "aqta' spazji miż-żewġ naħat ta'";
Blockly.Msg["TEXT_TRIM_OPERATOR_LEFT"] = "aqta' spazji mix-xellug ta'";
Blockly.Msg["TEXT_TRIM_OPERATOR_RIGHT"] = "aqta' spazji mil-lemin ta'";
Blockly.Msg["TEXT_TRIM_TOOLTIP"] = "Iġi b'kopja tat-test b'spazji mneħħija minn wieħed jew miż-żewġt itruf.";
Blockly.Msg["TODAY"] = "Illum";
Blockly.Msg["UNDO"] = "Mur Lura";
Blockly.Msg["UNKNOWN"] = "Mhux magħruf";
Blockly.Msg["UNNAMED_KEY"] = "bla isem";
Blockly.Msg["VARIABLES_DEFAULT_NAME"] = "oġġett";
Blockly.Msg["VARIABLES_GET_CREATE_SET"] = "Oħloq 'ranġa %1'";
Blockly.Msg["VARIABLES_GET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#get";  // untranslated
Blockly.Msg["VARIABLES_GET_TOOLTIP"] = "Tiġi bil-valur ta' dil-varjabbli.";
Blockly.Msg["VARIABLES_SET"] = "irranġa %1 għal %2";
Blockly.Msg["VARIABLES_SET_CREATE_GET"] = "Oħloq 'ikseb %1'";
Blockly.Msg["VARIABLES_SET_HELPURL"] = "https://github.com/RaspberryPiFoundation/blockly/wiki/Variables#set";  // untranslated
Blockly.Msg["VARIABLES_SET_TOOLTIP"] = "Tirranġa dil-varjabbli biex tiddaqqas għad-daħla.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS"] = "Varjabbli bl-isem '%1' ġa jeżisti.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE"] = "Varjabbli bl-isem '%1' ġa jeżisti għal tip ieħor: '%2'.";
Blockly.Msg["VARIABLE_ALREADY_EXISTS_FOR_A_PARAMETER"] = "Varjabbli bl-isem '%1' ġa jeżisti bħala parametru fil-proċedura '%2'.";
Blockly.Msg["WINDOWS"] = "Windows";
Blockly.Msg["WORKSPACE_COMMENT_DEFAULT_TEXT"] = "Għid xi ħaġa...";
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_MANY"] = "%1 stacks of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ONE"] = "One stack of blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_BLOCKS_ZERO"] = "No blocks%2 in workspace.";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_MANY"] = " and %1 comments";  // untranslated
Blockly.Msg["WORKSPACE_CONTENTS_COMMENTS_ONE"] = " and one comment";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_1_STACK"] = "Blocks workspace. 1 stack of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_FLYOUT_WORKSPACE"] = "%1 blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MANY_STACKS"] = "Blocks workspace. %1 stacks of blocks";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_MUTATOR_WORKSPACE"] = "Block editor workspace";  // untranslated
Blockly.Msg["WORKSPACE_LABEL_PLAIN"] = "Blocks workspace.";  // untranslated
Blockly.Msg["ZOOM_IN"] = "Zoom in";  // untranslated
Blockly.Msg["ZOOM_OUT"] = "Zoom out";  // untranslated
Blockly.Msg["CONTROLS_FOREACH_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_FOR_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_IF_ELSEIF_TITLE_ELSEIF"] = Blockly.Msg["CONTROLS_IF_MSG_ELSEIF"];
Blockly.Msg["CONTROLS_IF_ELSE_TITLE_ELSE"] = Blockly.Msg["CONTROLS_IF_MSG_ELSE"];
Blockly.Msg["CONTROLS_IF_IF_TITLE_IF"] = Blockly.Msg["CONTROLS_IF_MSG_IF"];
Blockly.Msg["CONTROLS_IF_MSG_THEN"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["CONTROLS_WHILEUNTIL_INPUT_DO"] = Blockly.Msg["CONTROLS_REPEAT_INPUT_DO"];
Blockly.Msg["LISTS_CREATE_WITH_ITEM_TITLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["LISTS_GET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_GET_SUBLIST_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_INDEX_OF_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["LISTS_SET_INDEX_INPUT_IN_LIST"] = Blockly.Msg["LISTS_INLIST"];
Blockly.Msg["MATH_CHANGE_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["PROCEDURES_DEFRETURN_COMMENT"] = Blockly.Msg["PROCEDURES_DEFNORETURN_COMMENT"];
Blockly.Msg["PROCEDURES_DEFRETURN_DO"] = Blockly.Msg["PROCEDURES_DEFNORETURN_DO"];
Blockly.Msg["PROCEDURES_DEFRETURN_PROCEDURE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_PROCEDURE"];
Blockly.Msg["PROCEDURES_DEFRETURN_TITLE"] = Blockly.Msg["PROCEDURES_DEFNORETURN_TITLE"];
Blockly.Msg["TEXT_APPEND_VARIABLE"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];
Blockly.Msg["TEXT_CREATE_JOIN_ITEM_TITLE_ITEM"] = Blockly.Msg["VARIABLES_DEFAULT_NAME"];

Blockly.Msg["COLOUR_HUE"] = "20";
Blockly.Msg["LISTS_HUE"] = "260";
Blockly.Msg["LOGIC_HUE"] = "210";
Blockly.Msg["LOOPS_HUE"] = "120";
Blockly.Msg["MATH_HUE"] = "230";
Blockly.Msg["PROCEDURES_HUE"] = "290";
Blockly.Msg["TEXTS_HUE"] = "160";
Blockly.Msg["VARIABLES_DYNAMIC_HUE"] = "310";
Blockly.Msg["VARIABLES_HUE"] = "330";
return Blockly.Msg;
}));
